From 4086f8c349569926fd8fbe967ec24bdeabc79f95 Mon Sep 17 00:00:00 2001
From: Ismo Puustinen <ismo.puustinen@intel.com>
Date: Fri, 19 Feb 2016 07:18:25 -0800
Subject: [PATCH 1/6] acpi: added a custom DSDT file.

The file has fixed GPIO IRQ assignment and moved SPI devices to be under
the SPI bus in the ACPI definitions as assumed by ACPI version 5.

Upstream-status: Inappropriate, custom firmware

Signed-off-by: Ismo Puustinen <ismo.puustinen@intel.com>
Signed-off-by: Saul Wold <sgw@linux.intel.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 include/DSDT.hex | 1191 ++++++++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 1191 insertions(+)
 create mode 100644 include/DSDT.hex

diff --git a/include/DSDT.hex b/include/DSDT.hex
new file mode 100644
index 000000000000..b1e296048090
--- /dev/null
+++ b/include/DSDT.hex
@@ -0,0 +1,1191 @@
+/*
+ * Copyright (c) 2013 Intel Corporation. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/*
+ * 
+ * Intel ACPI Component Architecture
+ * ASL+ Optimizing Compiler version 20150515-64
+ * Copyright (c) 2000 - 2015 Intel Corporation
+ * 
+ * Compilation of "dsdt-fixed-spi.dsl" - Wed Sep 23 12:49:37 2015
+ * 
+ * C source code output
+ * AML code block contains 0x23F5 bytes
+ *
+ */
+unsigned char AmlCode[] =
+{
+    0x44,0x53,0x44,0x54,0xF5,0x23,0x00,0x00,  /* 00000000    "DSDT.#.." */
+    0x01,0xD3,0x49,0x4E,0x54,0x45,0x4C,0x20,  /* 00000008    "..INTEL " */
+    0x51,0x75,0x61,0x72,0x6B,0x4E,0x63,0x53,  /* 00000010    "QuarkNcS" */
+    0x03,0x00,0x00,0x00,0x49,0x4E,0x54,0x4C,  /* 00000018    "....INTL" */
+    0x15,0x05,0x15,0x20,0x08,0x47,0x50,0x49,  /* 00000020    "... .GPI" */
+    0x43,0x00,0x5B,0x80,0x44,0x42,0x47,0x30,  /* 00000028    "C.[.DBG0" */
+    0x01,0x0A,0x80,0x01,0x5B,0x81,0x0B,0x44,  /* 00000030    "....[..D" */
+    0x42,0x47,0x30,0x01,0x49,0x4F,0x38,0x30,  /* 00000038    "BG0.IO80" */
+    0x08,0x5B,0x80,0x41,0x43,0x4D,0x53,0x01,  /* 00000040    ".[.ACMS." */
+    0x0A,0x72,0x0A,0x02,0x5B,0x81,0x10,0x41,  /* 00000048    ".r..[..A" */
+    0x43,0x4D,0x53,0x01,0x49,0x4E,0x44,0x58,  /* 00000050    "CMS.INDX" */
+    0x08,0x44,0x41,0x54,0x41,0x08,0x5B,0x80,  /* 00000058    ".DATA.[." */
+    0x4D,0x4E,0x56,0x53,0x00,0x0C,0x90,0x8C,  /* 00000060    "MNVS...." */
+    0x1C,0x0F,0x0B,0x00,0x02,0x5B,0x81,0x45,  /* 00000068    ".....[.E" */
+    0x07,0x4D,0x4E,0x56,0x53,0x01,0x4F,0x53,  /* 00000070    ".MNVS.OS" */
+    0x54,0x50,0x20,0x43,0x46,0x47,0x44,0x20,  /* 00000078    "TP CFGD " */
+    0x48,0x50,0x45,0x41,0x20,0x50,0x31,0x42,  /* 00000080    "HPEA P1B" */
+    0x42,0x20,0x50,0x42,0x41,0x42,0x20,0x47,  /* 00000088    "B PBAB G" */
+    0x50,0x30,0x42,0x20,0x47,0x50,0x41,0x42,  /* 00000090    "P0B GPAB" */
+    0x20,0x53,0x4D,0x42,0x42,0x20,0x4E,0x52,  /* 00000098    " SMBB NR" */
+    0x56,0x31,0x20,0x57,0x44,0x54,0x42,0x20,  /* 000000A0    "V1 WDTB " */
+    0x48,0x50,0x54,0x42,0x20,0x48,0x50,0x54,  /* 000000A8    "HPTB HPT" */
+    0x53,0x20,0x50,0x45,0x58,0x42,0x20,0x50,  /* 000000B0    "S PEXB P" */
+    0x45,0x58,0x53,0x20,0x52,0x43,0x42,0x42,  /* 000000B8    "EXS RCBB" */
+    0x20,0x52,0x43,0x42,0x53,0x20,0x41,0x50,  /* 000000C0    " RCBS AP" */
+    0x43,0x42,0x20,0x41,0x50,0x43,0x53,0x20,  /* 000000C8    "CB APCS " */
+    0x54,0x50,0x4D,0x50,0x20,0x44,0x42,0x47,  /* 000000D0    "TPMP DBG" */
+    0x50,0x20,0x50,0x54,0x59,0x50,0x20,0x41,  /* 000000D8    "P PTYP A" */
+    0x4C,0x54,0x53,0x20,0x5B,0x80,0x47,0x50,  /* 000000E0    "LTS [.GP" */
+    0x45,0x42,0x01,0x0B,0x00,0x11,0x0A,0x40,  /* 000000E8    "EB.....@" */
+    0x5B,0x81,0x13,0x47,0x50,0x45,0x42,0x00,  /* 000000F0    "[..GPEB." */
+    0x00,0x40,0x08,0x53,0x4D,0x49,0x45,0x20,  /* 000000F8    ".@.SMIE " */
+    0x53,0x4D,0x49,0x53,0x20,0x10,0x12,0x5F,  /* 00000100    "SMIS .._" */
+    0x50,0x52,0x5F,0x5B,0x83,0x0B,0x43,0x50,  /* 00000108    "PR_[..CP" */
+    0x55,0x30,0x01,0x10,0x10,0x00,0x00,0x06,  /* 00000110    "U0......" */
+    0x08,0x5F,0x53,0x30,0x5F,0x12,0x06,0x04,  /* 00000118    "._S0_..." */
+    0x00,0x00,0x00,0x00,0x08,0x5F,0x53,0x33,  /* 00000120    "....._S3" */
+    0x5F,0x12,0x07,0x04,0x0A,0x05,0x00,0x00,  /* 00000128    "_......." */
+    0x00,0x08,0x5F,0x53,0x34,0x5F,0x12,0x07,  /* 00000130    ".._S4_.." */
+    0x04,0x0A,0x06,0x00,0x00,0x00,0x08,0x5F,  /* 00000138    "......._" */
+    0x53,0x35,0x5F,0x12,0x07,0x04,0x0A,0x07,  /* 00000140    "S5_....." */
+    0x00,0x00,0x00,0x10,0x4E,0x12,0x5F,0x47,  /* 00000148    "....N._G" */
+    0x50,0x45,0x14,0x13,0x5F,0x4C,0x30,0x44,  /* 00000150    "PE.._L0D" */
+    0x00,0x86,0x5C,0x2E,0x5F,0x53,0x42,0x5F,  /* 00000158    "..\._SB_" */
+    0x53,0x4C,0x50,0x42,0x0A,0x02,0x14,0x13,  /* 00000160    "SLPB...." */
+    0x5F,0x4C,0x30,0x45,0x00,0x86,0x5C,0x2E,  /* 00000168    "_L0E..\." */
+    0x5F,0x53,0x42,0x5F,0x50,0x57,0x52,0x42,  /* 00000170    "_SB_PWRB" */
+    0x0A,0x02,0x14,0x4D,0x0C,0x5F,0x4C,0x30,  /* 00000178    "...M._L0" */
+    0x46,0x00,0x86,0x5C,0x2F,0x03,0x5F,0x53,  /* 00000180    "F..\/._S" */
+    0x42,0x5F,0x50,0x43,0x49,0x30,0x53,0x44,  /* 00000188    "B_PCI0SD" */
+    0x49,0x4F,0x0A,0x02,0x86,0x5C,0x2F,0x03,  /* 00000190    "IO...\/." */
+    0x5F,0x53,0x42,0x5F,0x50,0x43,0x49,0x30,  /* 00000198    "_SB_PCI0" */
+    0x55,0x52,0x54,0x30,0x0A,0x02,0x86,0x5C,  /* 000001A0    "URT0...\" */
+    0x2F,0x03,0x5F,0x53,0x42,0x5F,0x50,0x43,  /* 000001A8    "/._SB_PC" */
+    0x49,0x30,0x55,0x53,0x42,0x44,0x0A,0x02,  /* 000001B0    "I0USBD.." */
+    0x86,0x5C,0x2F,0x03,0x5F,0x53,0x42,0x5F,  /* 000001B8    ".\/._SB_" */
+    0x50,0x43,0x49,0x30,0x45,0x48,0x43,0x49,  /* 000001C0    "PCI0EHCI" */
+    0x0A,0x02,0x86,0x5C,0x2F,0x03,0x5F,0x53,  /* 000001C8    "...\/._S" */
+    0x42,0x5F,0x50,0x43,0x49,0x30,0x4F,0x48,  /* 000001D0    "B_PCI0OH" */
+    0x43,0x49,0x0A,0x02,0x86,0x5C,0x2F,0x03,  /* 000001D8    "CI...\/." */
+    0x5F,0x53,0x42,0x5F,0x50,0x43,0x49,0x30,  /* 000001E0    "_SB_PCI0" */
+    0x55,0x52,0x54,0x31,0x0A,0x02,0x86,0x5C,  /* 000001E8    "URT1...\" */
+    0x2F,0x03,0x5F,0x53,0x42,0x5F,0x50,0x43,  /* 000001F0    "/._SB_PC" */
+    0x49,0x30,0x45,0x4E,0x54,0x30,0x0A,0x02,  /* 000001F8    "I0ENT0.." */
+    0x86,0x5C,0x2F,0x03,0x5F,0x53,0x42,0x5F,  /* 00000200    ".\/._SB_" */
+    0x50,0x43,0x49,0x30,0x45,0x4E,0x54,0x31,  /* 00000208    "PCI0ENT1" */
+    0x0A,0x02,0x86,0x5C,0x2F,0x03,0x5F,0x53,  /* 00000210    "...\/._S" */
+    0x42,0x5F,0x50,0x43,0x49,0x30,0x53,0x50,  /* 00000218    "B_PCI0SP" */
+    0x49,0x30,0x0A,0x02,0x86,0x5C,0x2F,0x03,  /* 00000220    "I0...\/." */
+    0x5F,0x53,0x42,0x5F,0x50,0x43,0x49,0x30,  /* 00000228    "_SB_PCI0" */
+    0x53,0x50,0x49,0x31,0x0A,0x02,0x86,0x5C,  /* 00000230    "SPI1...\" */
+    0x2F,0x03,0x5F,0x53,0x42,0x5F,0x50,0x43,  /* 00000238    "/._SB_PC" */
+    0x49,0x30,0x47,0x49,0x50,0x30,0x0A,0x02,  /* 00000240    "I0GIP0.." */
+    0x14,0x06,0x5F,0x4C,0x31,0x30,0x00,0x14,  /* 00000248    ".._L10.." */
+    0x2A,0x5F,0x4C,0x31,0x31,0x00,0x86,0x5C,  /* 00000250    "*_L11..\" */
+    0x2F,0x03,0x5F,0x53,0x42,0x5F,0x50,0x43,  /* 00000258    "/._SB_PC" */
+    0x49,0x30,0x50,0x45,0x58,0x30,0x0A,0x02,  /* 00000260    "I0PEX0.." */
+    0x86,0x5C,0x2F,0x03,0x5F,0x53,0x42,0x5F,  /* 00000268    ".\/._SB_" */
+    0x50,0x43,0x49,0x30,0x50,0x45,0x58,0x31,  /* 00000270    "PCI0PEX1" */
+    0x0A,0x02,0x5B,0x82,0x23,0x2E,0x5F,0x53,  /* 00000278    "..[.#._S" */
+    0x42,0x5F,0x53,0x4C,0x50,0x42,0x08,0x5F,  /* 00000280    "B_SLPB._" */
+    0x48,0x49,0x44,0x0C,0x41,0xD0,0x0C,0x0E,  /* 00000288    "HID.A..." */
+    0x14,0x0E,0x5F,0x50,0x52,0x57,0x00,0xA4,  /* 00000290    ".._PRW.." */
+    0x12,0x06,0x02,0x0A,0x0D,0x0A,0x04,0x5B,  /* 00000298    ".......[" */
+    0x82,0x23,0x2E,0x5F,0x53,0x42,0x5F,0x50,  /* 000002A0    ".#._SB_P" */
+    0x57,0x52,0x42,0x08,0x5F,0x48,0x49,0x44,  /* 000002A8    "WRB._HID" */
+    0x0C,0x41,0xD0,0x0C,0x0C,0x14,0x0E,0x5F,  /* 000002B0    ".A....._" */
+    0x50,0x52,0x57,0x00,0xA4,0x12,0x06,0x02,  /* 000002B8    "PRW....." */
+    0x0A,0x0E,0x0A,0x04,0x14,0x08,0x5F,0x57,  /* 000002C0    "......_W" */
+    0x41,0x4B,0x09,0xA4,0x00,0x14,0x20,0x5F,  /* 000002C8    "AK.... _" */
+    0x50,0x54,0x53,0x01,0xA0,0x19,0x93,0x68,  /* 000002D0    "PTS....h" */
+    0x0A,0x03,0x70,0xFF,0x53,0x4D,0x49,0x53,  /* 000002D8    "..p.SMIS" */
+    0x70,0x53,0x4D,0x49,0x45,0x60,0x7D,0x60,  /* 000002E0    "pSMIE`}`" */
+    0x0A,0x04,0x53,0x4D,0x49,0x45,0x14,0x0C,  /* 000002E8    "..SMIE.." */
+    0x5F,0x50,0x49,0x43,0x01,0x70,0x68,0x47,  /* 000002F0    "_PIC.phG" */
+    0x50,0x49,0x43,0x10,0x89,0x0F,0x02,0x5F,  /* 000002F8    "PIC...._" */
+    0x53,0x42,0x5F,0x5B,0x82,0x80,0x44,0x01,  /* 00000300    "SB_[..D." */
+    0x50,0x43,0x49,0x30,0x08,0x5F,0x48,0x49,  /* 00000308    "PCI0._HI" */
+    0x44,0x0C,0x41,0xD0,0x0A,0x08,0x08,0x5F,  /* 00000310    "D.A...._" */
+    0x43,0x49,0x44,0x0C,0x41,0xD0,0x0A,0x03,  /* 00000318    "CID.A..." */
+    0x08,0x5F,0x41,0x44,0x52,0x00,0x14,0x4D,  /* 00000320    "._ADR..M" */
+    0x10,0x5F,0x49,0x4E,0x49,0x00,0x70,0x0A,  /* 00000328    "._INI.p." */
+    0xF0,0x4F,0x53,0x54,0x50,0xA0,0x4E,0x0F,  /* 00000330    ".OSTP.N." */
+    0x5B,0x12,0x5F,0x4F,0x53,0x49,0x60,0xA0,  /* 00000338    "[._OSI`." */
+    0x19,0x5F,0x4F,0x53,0x49,0x0D,0x57,0x69,  /* 00000340    "._OSI.Wi" */
+    0x6E,0x64,0x6F,0x77,0x73,0x20,0x32,0x30,  /* 00000348    "ndows 20" */
+    0x30,0x31,0x00,0x70,0x01,0x4F,0x53,0x54,  /* 00000350    "01.p.OST" */
+    0x50,0xA0,0x1E,0x5F,0x4F,0x53,0x49,0x0D,  /* 00000358    "P.._OSI." */
+    0x57,0x69,0x6E,0x64,0x6F,0x77,0x73,0x20,  /* 00000360    "Windows " */
+    0x32,0x30,0x30,0x31,0x20,0x53,0x50,0x31,  /* 00000368    "2001 SP1" */
+    0x00,0x70,0x0A,0x02,0x4F,0x53,0x54,0x50,  /* 00000370    ".p..OSTP" */
+    0xA0,0x1E,0x5F,0x4F,0x53,0x49,0x0D,0x57,  /* 00000378    ".._OSI.W" */
+    0x69,0x6E,0x64,0x6F,0x77,0x73,0x20,0x32,  /* 00000380    "indows 2" */
+    0x30,0x30,0x31,0x20,0x53,0x50,0x32,0x00,  /* 00000388    "001 SP2." */
+    0x70,0x0A,0x04,0x4F,0x53,0x54,0x50,0xA0,  /* 00000390    "p..OSTP." */
+    0x1C,0x5F,0x4F,0x53,0x49,0x0D,0x57,0x69,  /* 00000398    "._OSI.Wi" */
+    0x6E,0x64,0x6F,0x77,0x73,0x20,0x32,0x30,  /* 000003A0    "ndows 20" */
+    0x30,0x31,0x2E,0x31,0x00,0x70,0x0A,0x08,  /* 000003A8    "01.1.p.." */
+    0x4F,0x53,0x54,0x50,0xA0,0x1A,0x5F,0x4F,  /* 000003B0    "OSTP.._O" */
+    0x53,0x49,0x0D,0x57,0x69,0x6E,0x64,0x6F,  /* 000003B8    "SI.Windo" */
+    0x77,0x73,0x20,0x32,0x30,0x30,0x36,0x00,  /* 000003C0    "ws 2006." */
+    0x70,0x0A,0x10,0x4F,0x53,0x54,0x50,0xA0,  /* 000003C8    "p..OSTP." */
+    0x1A,0x5F,0x4F,0x53,0x49,0x0D,0x57,0x69,  /* 000003D0    "._OSI.Wi" */
+    0x6E,0x64,0x6F,0x77,0x73,0x20,0x32,0x30,  /* 000003D8    "ndows 20" */
+    0x30,0x39,0x00,0x70,0x0A,0x11,0x4F,0x53,  /* 000003E0    "09.p..OS" */
+    0x54,0x50,0xA0,0x1A,0x5F,0x4F,0x53,0x49,  /* 000003E8    "TP.._OSI" */
+    0x0D,0x57,0x69,0x6E,0x64,0x6F,0x77,0x73,  /* 000003F0    ".Windows" */
+    0x20,0x32,0x30,0x31,0x32,0x00,0x70,0x0A,  /* 000003F8    " 2012.p." */
+    0x12,0x4F,0x53,0x54,0x50,0xA0,0x1A,0x5F,  /* 00000400    ".OSTP.._" */
+    0x4F,0x53,0x49,0x0D,0x57,0x69,0x6E,0x64,  /* 00000408    "OSI.Wind" */
+    0x6F,0x77,0x73,0x20,0x32,0x30,0x31,0x33,  /* 00000410    "ows 2013" */
+    0x00,0x70,0x0A,0x13,0x4F,0x53,0x54,0x50,  /* 00000418    ".p..OSTP" */
+    0xA0,0x13,0x5F,0x4F,0x53,0x49,0x0D,0x4C,  /* 00000420    ".._OSI.L" */
+    0x69,0x6E,0x75,0x78,0x00,0x70,0x0A,0xF0,  /* 00000428    "inux.p.." */
+    0x4F,0x53,0x54,0x50,0x08,0x50,0x42,0x52,  /* 00000430    "OSTP.PBR" */
+    0x53,0x11,0x42,0x07,0x0A,0x6E,0x88,0x0D,  /* 00000438    "S.B..n.." */
+    0x00,0x02,0x0C,0x00,0x00,0x00,0x00,0x00,  /* 00000440    "........" */
+    0x1F,0x00,0x00,0x00,0x20,0x00,0x88,0x0D,  /* 00000448    ".... ..." */
+    0x00,0x01,0x0C,0x03,0x00,0x00,0x00,0x00,  /* 00000450    "........" */
+    0xF7,0x0C,0x00,0x00,0xF8,0x0C,0x47,0x01,  /* 00000458    "......G." */
+    0xF8,0x0C,0xF8,0x0C,0x01,0x08,0x88,0x0D,  /* 00000460    "........" */
+    0x00,0x01,0x0C,0x03,0x00,0x00,0x00,0x0D,  /* 00000468    "........" */
+    0xFF,0xFF,0x00,0x00,0x00,0xF3,0x87,0x17,  /* 00000470    "........" */
+    0x00,0x00,0x0C,0x03,0x00,0x00,0x00,0x00,  /* 00000478    "........" */
+    0x00,0x00,0x0A,0x00,0xFF,0xFF,0x0B,0x00,  /* 00000480    "........" */
+    0x00,0x00,0x00,0x00,0x00,0x00,0x02,0x00,  /* 00000488    "........" */
+    0x87,0x17,0x00,0x00,0x0C,0x03,0x00,0x00,  /* 00000490    "........" */
+    0x00,0x00,0x00,0x00,0x00,0x00,0xFF,0xFF,  /* 00000498    "........" */
+    0xBF,0xFE,0x00,0x00,0x00,0x00,0x00,0x00,  /* 000004A0    "........" */
+    0xC0,0xFE,0x79,0x00,0x14,0x4F,0x04,0x5F,  /* 000004A8    "..y..O._" */
+    0x43,0x52,0x53,0x00,0x8A,0x50,0x42,0x52,  /* 000004B0    "CRS..PBR" */
+    0x53,0x0A,0x5C,0x4D,0x4D,0x49,0x4E,0x8A,  /* 000004B8    "S.\MMIN." */
+    0x50,0x42,0x52,0x53,0x0A,0x60,0x4D,0x4D,  /* 000004C0    "PBRS.`MM" */
+    0x41,0x58,0x8A,0x50,0x42,0x52,0x53,0x0A,  /* 000004C8    "AX.PBRS." */
+    0x68,0x4D,0x4C,0x45,0x4E,0x7B,0x4D,0x4E,  /* 000004D0    "hMLEN{MN" */
+    0x52,0x44,0x0A,0x03,0x0A,0x08,0x0C,0x00,  /* 000004D8    "RD......" */
+    0xF0,0xFF,0xFF,0x4D,0x4D,0x49,0x4E,0x72,  /* 000004E0    "...MMINr" */
+    0x74,0x4D,0x4D,0x41,0x58,0x4D,0x4D,0x49,  /* 000004E8    "tMMAXMMI" */
+    0x4E,0x00,0x01,0x4D,0x4C,0x45,0x4E,0xA4,  /* 000004F0    "N..MLEN." */
+    0x50,0x42,0x52,0x53,0x5B,0x80,0x4D,0x4E,  /* 000004F8    "PBRS[.MN" */
+    0x52,0x5F,0x02,0x0A,0xD0,0x0A,0x10,0x5B,  /* 00000500    "R_.....[" */
+    0x81,0x10,0x4D,0x4E,0x52,0x5F,0x03,0x4D,  /* 00000508    "..MNR_.M" */
+    0x43,0x52,0x5F,0x20,0x4D,0x44,0x52,0x5F,  /* 00000510    "CR_ MDR_" */
+    0x20,0x14,0x25,0x4D,0x4E,0x52,0x44,0x0A,  /* 00000518    " .%MNRD." */
+    0x7D,0x79,0x68,0x0A,0x10,0x00,0x79,0x69,  /* 00000520    "}yh...yi" */
+    0x0A,0x08,0x00,0x60,0x7D,0x60,0x0C,0xF0,  /* 00000528    "...`}`.." */
+    0x00,0x00,0x10,0x60,0x70,0x60,0x4D,0x43,  /* 00000530    "...`p`MC" */
+    0x52,0x5F,0xA4,0x4D,0x44,0x52,0x5F,0x14,  /* 00000538    "R_.MDR_." */
+    0x26,0x4D,0x4E,0x57,0x52,0x0B,0x70,0x6A,  /* 00000540    "&MNWR.pj" */
+    0x4D,0x44,0x52,0x5F,0x7D,0x79,0x68,0x0A,  /* 00000548    "MDR_}yh." */
+    0x10,0x00,0x79,0x69,0x0A,0x08,0x00,0x60,  /* 00000550    "..yi...`" */
+    0x7D,0x60,0x0C,0xF0,0x00,0x00,0x11,0x60,  /* 00000558    "}`.....`" */
+    0x70,0x60,0x4D,0x43,0x52,0x5F,0x14,0x49,  /* 00000560    "p`MCR_.I" */
+    0x1C,0x5F,0x50,0x52,0x54,0x00,0xA0,0x47,  /* 00000568    "._PRT..G" */
+    0x11,0x93,0x47,0x50,0x49,0x43,0x00,0xA4,  /* 00000570    "..GPIC.." */
+    0x12,0x4D,0x10,0x0D,0x12,0x13,0x04,0x0C,  /* 00000578    ".M......" */
+    0xFF,0xFF,0x14,0x00,0x00,0x5E,0x2E,0x4C,  /* 00000580    ".....^.L" */
+    0x50,0x43,0x5F,0x4C,0x4E,0x4B,0x41,0x00,  /* 00000588    "PC_LNKA." */
+    0x12,0x13,0x04,0x0C,0xFF,0xFF,0x14,0x00,  /* 00000590    "........" */
+    0x01,0x5E,0x2E,0x4C,0x50,0x43,0x5F,0x4C,  /* 00000598    ".^.LPC_L" */
+    0x4E,0x4B,0x42,0x00,0x12,0x14,0x04,0x0C,  /* 000005A0    "NKB....." */
+    0xFF,0xFF,0x14,0x00,0x0A,0x02,0x5E,0x2E,  /* 000005A8    "......^." */
+    0x4C,0x50,0x43,0x5F,0x4C,0x4E,0x4B,0x43,  /* 000005B0    "LPC_LNKC" */
+    0x00,0x12,0x14,0x04,0x0C,0xFF,0xFF,0x14,  /* 000005B8    "........" */
+    0x00,0x0A,0x03,0x5E,0x2E,0x4C,0x50,0x43,  /* 000005C0    "...^.LPC" */
+    0x5F,0x4C,0x4E,0x4B,0x44,0x00,0x12,0x13,  /* 000005C8    "_LNKD..." */
+    0x04,0x0C,0xFF,0xFF,0x15,0x00,0x00,0x5E,  /* 000005D0    ".......^" */
+    0x2E,0x4C,0x50,0x43,0x5F,0x4C,0x4E,0x4B,  /* 000005D8    ".LPC_LNK" */
+    0x41,0x00,0x12,0x13,0x04,0x0C,0xFF,0xFF,  /* 000005E0    "A......." */
+    0x15,0x00,0x01,0x5E,0x2E,0x4C,0x50,0x43,  /* 000005E8    "...^.LPC" */
+    0x5F,0x4C,0x4E,0x4B,0x42,0x00,0x12,0x14,  /* 000005F0    "_LNKB..." */
+    0x04,0x0C,0xFF,0xFF,0x15,0x00,0x0A,0x02,  /* 000005F8    "........" */
+    0x5E,0x2E,0x4C,0x50,0x43,0x5F,0x4C,0x4E,  /* 00000600    "^.LPC_LN" */
+    0x4B,0x43,0x00,0x12,0x14,0x04,0x0C,0xFF,  /* 00000608    "KC......" */
+    0xFF,0x15,0x00,0x0A,0x03,0x5E,0x2E,0x4C,  /* 00000610    ".....^.L" */
+    0x50,0x43,0x5F,0x4C,0x4E,0x4B,0x44,0x00,  /* 00000618    "PC_LNKD." */
+    0x12,0x13,0x04,0x0C,0xFF,0xFF,0x17,0x00,  /* 00000620    "........" */
+    0x00,0x5E,0x2E,0x4C,0x50,0x43,0x5F,0x4C,  /* 00000628    ".^.LPC_L" */
+    0x4E,0x4B,0x45,0x00,0x12,0x13,0x04,0x0C,  /* 00000630    "NKE....." */
+    0xFF,0xFF,0x17,0x00,0x01,0x5E,0x2E,0x4C,  /* 00000638    ".....^.L" */
+    0x50,0x43,0x5F,0x4C,0x4E,0x4B,0x46,0x00,  /* 00000640    "PC_LNKF." */
+    0x12,0x14,0x04,0x0C,0xFF,0xFF,0x17,0x00,  /* 00000648    "........" */
+    0x0A,0x02,0x5E,0x2E,0x4C,0x50,0x43,0x5F,  /* 00000650    "..^.LPC_" */
+    0x4C,0x4E,0x4B,0x47,0x00,0x12,0x14,0x04,  /* 00000658    "LNKG...." */
+    0x0C,0xFF,0xFF,0x17,0x00,0x0A,0x03,0x5E,  /* 00000660    ".......^" */
+    0x2E,0x4C,0x50,0x43,0x5F,0x4C,0x4E,0x4B,  /* 00000668    ".LPC_LNK" */
+    0x48,0x00,0x12,0x13,0x04,0x0C,0xFF,0xFF,  /* 00000670    "H......." */
+    0x1F,0x00,0x00,0x5E,0x2E,0x4C,0x50,0x43,  /* 00000678    "...^.LPC" */
+    0x5F,0x4C,0x4E,0x4B,0x41,0x00,0xA1,0x49,  /* 00000680    "_LNKA..I" */
+    0x0A,0xA4,0x12,0x45,0x0A,0x0D,0x12,0x0B,  /* 00000688    "...E...." */
+    0x04,0x0C,0xFF,0xFF,0x14,0x00,0x00,0x00,  /* 00000690    "........" */
+    0x0A,0x10,0x12,0x0B,0x04,0x0C,0xFF,0xFF,  /* 00000698    "........" */
+    0x14,0x00,0x01,0x00,0x0A,0x11,0x12,0x0C,  /* 000006A0    "........" */
+    0x04,0x0C,0xFF,0xFF,0x14,0x00,0x0A,0x02,  /* 000006A8    "........" */
+    0x00,0x0A,0x12,0x12,0x0C,0x04,0x0C,0xFF,  /* 000006B0    "........" */
+    0xFF,0x14,0x00,0x0A,0x03,0x00,0x0A,0x13,  /* 000006B8    "........" */
+    0x12,0x0B,0x04,0x0C,0xFF,0xFF,0x15,0x00,  /* 000006C0    "........" */
+    0x00,0x00,0x0A,0x10,0x12,0x0B,0x04,0x0C,  /* 000006C8    "........" */
+    0xFF,0xFF,0x15,0x00,0x01,0x00,0x0A,0x11,  /* 000006D0    "........" */
+    0x12,0x0C,0x04,0x0C,0xFF,0xFF,0x15,0x00,  /* 000006D8    "........" */
+    0x0A,0x02,0x00,0x0A,0x12,0x12,0x0C,0x04,  /* 000006E0    "........" */
+    0x0C,0xFF,0xFF,0x15,0x00,0x0A,0x03,0x00,  /* 000006E8    "........" */
+    0x0A,0x13,0x12,0x0B,0x04,0x0C,0xFF,0xFF,  /* 000006F0    "........" */
+    0x17,0x00,0x00,0x00,0x0A,0x14,0x12,0x0B,  /* 000006F8    "........" */
+    0x04,0x0C,0xFF,0xFF,0x17,0x00,0x01,0x00,  /* 00000700    "........" */
+    0x0A,0x15,0x12,0x0C,0x04,0x0C,0xFF,0xFF,  /* 00000708    "........" */
+    0x17,0x00,0x0A,0x02,0x00,0x0A,0x16,0x12,  /* 00000710    "........" */
+    0x0C,0x04,0x0C,0xFF,0xFF,0x17,0x00,0x0A,  /* 00000718    "........" */
+    0x03,0x00,0x0A,0x17,0x12,0x0B,0x04,0x0C,  /* 00000720    "........" */
+    0xFF,0xFF,0x1F,0x00,0x00,0x00,0x0A,0x10,  /* 00000728    "........" */
+    0x5B,0x82,0x45,0x0B,0x49,0x4F,0x43,0x4D,  /* 00000730    "[.E.IOCM" */
+    0x08,0x5F,0x48,0x49,0x44,0x0C,0x41,0xD0,  /* 00000738    "._HID.A." */
+    0x0C,0x02,0x08,0x43,0x52,0x53,0x5F,0x11,  /* 00000740    "...CRS_." */
+    0x42,0x04,0x0A,0x3E,0x86,0x09,0x00,0x00,  /* 00000748    "B..>...." */
+    0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,  /* 00000750    "........" */
+    0x86,0x09,0x00,0x00,0x00,0x00,0x00,0x00,  /* 00000758    "........" */
+    0x00,0x00,0x00,0x00,0x86,0x09,0x00,0x00,  /* 00000760    "........" */
+    0x00,0x00,0x0C,0x00,0x00,0x00,0x02,0x00,  /* 00000768    "........" */
+    0x86,0x09,0x00,0x00,0x00,0x00,0x0E,0x00,  /* 00000770    "........" */
+    0x00,0x00,0x02,0x00,0x86,0x09,0x00,0x00,  /* 00000778    "........" */
+    0x00,0x00,0x80,0xFF,0x00,0x00,0x80,0x00,  /* 00000780    "........" */
+    0x79,0x00,0x14,0x4C,0x05,0x5F,0x43,0x52,  /* 00000788    "y..L._CR" */
+    0x53,0x00,0x8A,0x43,0x52,0x53,0x5F,0x0A,  /* 00000790    "S..CRS_." */
+    0x04,0x4D,0x42,0x52,0x30,0x70,0x50,0x45,  /* 00000798    ".MBR0pPE" */
+    0x58,0x42,0x4D,0x42,0x52,0x30,0x8A,0x43,  /* 000007A0    "XBMBR0.C" */
+    0x52,0x53,0x5F,0x0A,0x08,0x4D,0x42,0x52,  /* 000007A8    "RS_..MBR" */
+    0x31,0x70,0x50,0x45,0x58,0x53,0x4D,0x42,  /* 000007B0    "1pPEXSMB" */
+    0x52,0x31,0x8A,0x43,0x52,0x53,0x5F,0x0A,  /* 000007B8    "R1.CRS_." */
+    0x10,0x4D,0x42,0x52,0x32,0x70,0x52,0x43,  /* 000007C0    ".MBR2pRC" */
+    0x42,0x42,0x4D,0x42,0x52,0x32,0x8A,0x43,  /* 000007C8    "BBMBR2.C" */
+    0x52,0x53,0x5F,0x0A,0x14,0x4D,0x42,0x52,  /* 000007D0    "RS_..MBR" */
+    0x33,0x70,0x52,0x43,0x42,0x53,0x4D,0x42,  /* 000007D8    "3pRCBSMB" */
+    0x52,0x33,0xA4,0x43,0x52,0x53,0x5F,0x5B,  /* 000007E0    "R3.CRS_[" */
+    0x82,0x45,0x10,0x50,0x45,0x58,0x30,0x08,  /* 000007E8    ".E.PEX0." */
+    0x5F,0x41,0x44,0x52,0x0C,0x00,0x00,0x17,  /* 000007F0    "_ADR...." */
+    0x00,0x08,0x5F,0x50,0x52,0x57,0x12,0x06,  /* 000007F8    ".._PRW.." */
+    0x02,0x0A,0x11,0x0A,0x03,0x5B,0x80,0x50,  /* 00000800    ".....[.P" */
+    0x45,0x53,0x30,0x02,0x0A,0x40,0x0A,0xA0,  /* 00000808    "ES0..@.." */
+    0x5B,0x81,0x48,0x04,0x50,0x45,0x53,0x30,  /* 00000810    "[.H.PES0" */
+    0x00,0x00,0x40,0x0D,0x41,0x42,0x50,0x30,  /* 00000818    "..@.ABP0" */
+    0x01,0x00,0x02,0x50,0x44,0x43,0x30,0x01,  /* 00000820    "...PDC0." */
+    0x00,0x02,0x50,0x44,0x53,0x30,0x01,0x00,  /* 00000828    "..PDS0.." */
+    0x01,0x4C,0x53,0x43,0x30,0x01,0x00,0x27,  /* 00000830    ".LSC0..'" */
+    0x00,0x10,0x50,0x4D,0x53,0x30,0x01,0x00,  /* 00000838    "..PMS0.." */
+    0x4F,0x3A,0x00,0x1E,0x48,0x50,0x45,0x30,  /* 00000840    "O:..HPE0" */
+    0x01,0x50,0x43,0x45,0x30,0x01,0x00,0x1E,  /* 00000848    ".PCE0..." */
+    0x48,0x50,0x53,0x30,0x01,0x50,0x43,0x53,  /* 00000850    "HPS0.PCS" */
+    0x30,0x01,0x14,0x43,0x09,0x5F,0x50,0x52,  /* 00000858    "0..C._PR" */
+    0x54,0x00,0xA0,0x4B,0x05,0x93,0x47,0x50,  /* 00000860    "T..K..GP" */
+    0x49,0x43,0x00,0xA4,0x12,0x41,0x05,0x04,  /* 00000868    "IC...A.." */
+    0x12,0x12,0x04,0x0B,0xFF,0xFF,0x00,0x5E,  /* 00000870    ".......^" */
+    0x5E,0x2E,0x4C,0x50,0x43,0x5F,0x4C,0x4E,  /* 00000878    "^.LPC_LN" */
+    0x4B,0x45,0x00,0x12,0x12,0x04,0x0B,0xFF,  /* 00000880    "KE......" */
+    0xFF,0x01,0x5E,0x5E,0x2E,0x4C,0x50,0x43,  /* 00000888    "..^^.LPC" */
+    0x5F,0x4C,0x4E,0x4B,0x46,0x00,0x12,0x13,  /* 00000890    "_LNKF..." */
+    0x04,0x0B,0xFF,0xFF,0x0A,0x02,0x5E,0x5E,  /* 00000898    "......^^" */
+    0x2E,0x4C,0x50,0x43,0x5F,0x4C,0x4E,0x4B,  /* 000008A0    ".LPC_LNK" */
+    0x47,0x00,0x12,0x13,0x04,0x0B,0xFF,0xFF,  /* 000008A8    "G......." */
+    0x0A,0x03,0x5E,0x5E,0x2E,0x4C,0x50,0x43,  /* 000008B0    "..^^.LPC" */
+    0x5F,0x4C,0x4E,0x4B,0x48,0x00,0xA1,0x2F,  /* 000008B8    "_LNKH../" */
+    0xA4,0x12,0x2C,0x04,0x12,0x09,0x04,0x0B,  /* 000008C0    "..,....." */
+    0xFF,0xFF,0x00,0x00,0x0A,0x14,0x12,0x09,  /* 000008C8    "........" */
+    0x04,0x0B,0xFF,0xFF,0x01,0x00,0x0A,0x15,  /* 000008D0    "........" */
+    0x12,0x0A,0x04,0x0B,0xFF,0xFF,0x0A,0x02,  /* 000008D8    "........" */
+    0x00,0x0A,0x16,0x12,0x0A,0x04,0x0B,0xFF,  /* 000008E0    "........" */
+    0xFF,0x0A,0x03,0x00,0x0A,0x17,0x5B,0x82,  /* 000008E8    "......[." */
+    0x45,0x10,0x50,0x45,0x58,0x31,0x08,0x5F,  /* 000008F0    "E.PEX1._" */
+    0x41,0x44,0x52,0x0C,0x01,0x00,0x17,0x00,  /* 000008F8    "ADR....." */
+    0x08,0x5F,0x50,0x52,0x57,0x12,0x06,0x02,  /* 00000900    "._PRW..." */
+    0x0A,0x11,0x0A,0x03,0x5B,0x80,0x50,0x45,  /* 00000908    "....[.PE" */
+    0x53,0x31,0x02,0x0A,0x40,0x0A,0xA0,0x5B,  /* 00000910    "S1..@..[" */
+    0x81,0x48,0x04,0x50,0x45,0x53,0x31,0x00,  /* 00000918    ".H.PES1." */
+    0x00,0x40,0x0D,0x41,0x42,0x50,0x31,0x01,  /* 00000920    ".@.ABP1." */
+    0x00,0x02,0x50,0x44,0x43,0x31,0x01,0x00,  /* 00000928    "..PDC1.." */
+    0x02,0x50,0x44,0x53,0x31,0x01,0x00,0x01,  /* 00000930    ".PDS1..." */
+    0x4C,0x53,0x43,0x31,0x01,0x00,0x27,0x00,  /* 00000938    "LSC1..'." */
+    0x10,0x50,0x4D,0x53,0x31,0x01,0x00,0x4F,  /* 00000940    ".PMS1..O" */
+    0x3A,0x00,0x1E,0x48,0x50,0x45,0x31,0x01,  /* 00000948    ":..HPE1." */
+    0x50,0x43,0x45,0x31,0x01,0x00,0x1E,0x48,  /* 00000950    "PCE1...H" */
+    0x50,0x53,0x31,0x01,0x50,0x43,0x53,0x31,  /* 00000958    "PS1.PCS1" */
+    0x01,0x14,0x43,0x09,0x5F,0x50,0x52,0x54,  /* 00000960    "..C._PRT" */
+    0x00,0xA0,0x4B,0x05,0x93,0x47,0x50,0x49,  /* 00000968    "..K..GPI" */
+    0x43,0x00,0xA4,0x12,0x41,0x05,0x04,0x12,  /* 00000970    "C...A..." */
+    0x12,0x04,0x0B,0xFF,0xFF,0x00,0x5E,0x5E,  /* 00000978    "......^^" */
+    0x2E,0x4C,0x50,0x43,0x5F,0x4C,0x4E,0x4B,  /* 00000980    ".LPC_LNK" */
+    0x46,0x00,0x12,0x12,0x04,0x0B,0xFF,0xFF,  /* 00000988    "F......." */
+    0x01,0x5E,0x5E,0x2E,0x4C,0x50,0x43,0x5F,  /* 00000990    ".^^.LPC_" */
+    0x4C,0x4E,0x4B,0x47,0x00,0x12,0x13,0x04,  /* 00000998    "LNKG...." */
+    0x0B,0xFF,0xFF,0x0A,0x02,0x5E,0x5E,0x2E,  /* 000009A0    ".....^^." */
+    0x4C,0x50,0x43,0x5F,0x4C,0x4E,0x4B,0x48,  /* 000009A8    "LPC_LNKH" */
+    0x00,0x12,0x13,0x04,0x0B,0xFF,0xFF,0x0A,  /* 000009B0    "........" */
+    0x03,0x5E,0x5E,0x2E,0x4C,0x50,0x43,0x5F,  /* 000009B8    ".^^.LPC_" */
+    0x4C,0x4E,0x4B,0x45,0x00,0xA1,0x2F,0xA4,  /* 000009C0    "LNKE../." */
+    0x12,0x2C,0x04,0x12,0x09,0x04,0x0B,0xFF,  /* 000009C8    ".,......" */
+    0xFF,0x00,0x00,0x0A,0x15,0x12,0x09,0x04,  /* 000009D0    "........" */
+    0x0B,0xFF,0xFF,0x01,0x00,0x0A,0x16,0x12,  /* 000009D8    "........" */
+    0x0A,0x04,0x0B,0xFF,0xFF,0x0A,0x02,0x00,  /* 000009E0    "........" */
+    0x0A,0x17,0x12,0x0A,0x04,0x0B,0xFF,0xFF,  /* 000009E8    "........" */
+    0x0A,0x03,0x00,0x0A,0x14,0x5B,0x82,0x22,  /* 000009F0    ".....[."" */
+    0x53,0x44,0x49,0x4F,0x08,0x5F,0x41,0x44,  /* 000009F8    "SDIO._AD" */
+    0x52,0x0C,0x00,0x00,0x14,0x00,0x08,0x5F,  /* 00000A00    "R......_" */
+    0x53,0x54,0x41,0x0A,0x0F,0x08,0x5F,0x50,  /* 00000A08    "STA..._P" */
+    0x52,0x57,0x12,0x06,0x02,0x0A,0x0F,0x0A,  /* 00000A10    "RW......" */
+    0x03,0x5B,0x82,0x22,0x55,0x52,0x54,0x30,  /* 00000A18    ".[."URT0" */
+    0x08,0x5F,0x41,0x44,0x52,0x0C,0x01,0x00,  /* 00000A20    "._ADR..." */
+    0x14,0x00,0x08,0x5F,0x53,0x54,0x41,0x0A,  /* 00000A28    "..._STA." */
+    0x0F,0x08,0x5F,0x50,0x52,0x57,0x12,0x06,  /* 00000A30    ".._PRW.." */
+    0x02,0x0A,0x0F,0x0A,0x03,0x5B,0x82,0x22,  /* 00000A38    ".....[."" */
+    0x55,0x53,0x42,0x44,0x08,0x5F,0x41,0x44,  /* 00000A40    "USBD._AD" */
+    0x52,0x0C,0x02,0x00,0x14,0x00,0x08,0x5F,  /* 00000A48    "R......_" */
+    0x53,0x54,0x41,0x0A,0x0F,0x08,0x5F,0x50,  /* 00000A50    "STA..._P" */
+    0x52,0x57,0x12,0x06,0x02,0x0A,0x0F,0x0A,  /* 00000A58    "RW......" */
+    0x03,0x5B,0x82,0x22,0x45,0x48,0x43,0x49,  /* 00000A60    ".[."EHCI" */
+    0x08,0x5F,0x41,0x44,0x52,0x0C,0x03,0x00,  /* 00000A68    "._ADR..." */
+    0x14,0x00,0x08,0x5F,0x53,0x54,0x41,0x0A,  /* 00000A70    "..._STA." */
+    0x0F,0x08,0x5F,0x50,0x52,0x57,0x12,0x06,  /* 00000A78    ".._PRW.." */
+    0x02,0x0A,0x0F,0x0A,0x03,0x5B,0x82,0x22,  /* 00000A80    ".....[."" */
+    0x4F,0x48,0x43,0x49,0x08,0x5F,0x41,0x44,  /* 00000A88    "OHCI._AD" */
+    0x52,0x0C,0x04,0x00,0x14,0x00,0x08,0x5F,  /* 00000A90    "R......_" */
+    0x53,0x54,0x41,0x0A,0x0F,0x08,0x5F,0x50,  /* 00000A98    "STA..._P" */
+    0x52,0x57,0x12,0x06,0x02,0x0A,0x0F,0x0A,  /* 00000AA0    "RW......" */
+    0x03,0x5B,0x82,0x22,0x55,0x52,0x54,0x31,  /* 00000AA8    ".[."URT1" */
+    0x08,0x5F,0x41,0x44,0x52,0x0C,0x05,0x00,  /* 00000AB0    "._ADR..." */
+    0x14,0x00,0x08,0x5F,0x53,0x54,0x41,0x0A,  /* 00000AB8    "..._STA." */
+    0x0F,0x08,0x5F,0x50,0x52,0x57,0x12,0x06,  /* 00000AC0    ".._PRW.." */
+    0x02,0x0A,0x0F,0x0A,0x03,0x5B,0x82,0x22,  /* 00000AC8    ".....[."" */
+    0x45,0x4E,0x54,0x30,0x08,0x5F,0x41,0x44,  /* 00000AD0    "ENT0._AD" */
+    0x52,0x0C,0x06,0x00,0x14,0x00,0x08,0x5F,  /* 00000AD8    "R......_" */
+    0x53,0x54,0x41,0x0A,0x0F,0x08,0x5F,0x50,  /* 00000AE0    "STA..._P" */
+    0x52,0x57,0x12,0x06,0x02,0x0A,0x0F,0x0A,  /* 00000AE8    "RW......" */
+    0x03,0x5B,0x82,0x22,0x45,0x4E,0x54,0x31,  /* 00000AF0    ".[."ENT1" */
+    0x08,0x5F,0x41,0x44,0x52,0x0C,0x07,0x00,  /* 00000AF8    "._ADR..." */
+    0x14,0x00,0x08,0x5F,0x53,0x54,0x41,0x0A,  /* 00000B00    "..._STA." */
+    0x0F,0x08,0x5F,0x50,0x52,0x57,0x12,0x06,  /* 00000B08    ".._PRW.." */
+    0x02,0x0A,0x0F,0x0A,0x03,0x5B,0x82,0x4A,  /* 00000B10    ".....[.J" */
+    0x16,0x53,0x50,0x49,0x30,0x08,0x5F,0x41,  /* 00000B18    ".SPI0._A" */
+    0x44,0x52,0x0C,0x00,0x00,0x15,0x00,0x08,  /* 00000B20    "DR......" */
+    0x5F,0x53,0x54,0x41,0x0A,0x0F,0x08,0x5F,  /* 00000B28    "_STA..._" */
+    0x50,0x52,0x57,0x12,0x06,0x02,0x0A,0x0F,  /* 00000B30    "PRW....." */
+    0x0A,0x03,0x5B,0x82,0x42,0x0A,0x41,0x44,  /* 00000B38    "..[.B.AD" */
+    0x43,0x31,0x08,0x5F,0x48,0x49,0x44,0x0D,  /* 00000B40    "C1._HID." */
+    0x49,0x4E,0x54,0x33,0x34,0x39,0x34,0x00,  /* 00000B48    "INT3494." */
+    0x08,0x5F,0x43,0x49,0x44,0x0D,0x49,0x4E,  /* 00000B50    "._CID.IN" */
+    0x54,0x33,0x34,0x39,0x34,0x00,0x08,0x52,  /* 00000B58    "T3494..R" */
+    0x42,0x55,0x46,0x11,0x48,0x05,0x0A,0x54,  /* 00000B60    "BUF.H..T" */
+    0x8E,0x22,0x00,0x01,0x00,0x02,0x02,0x00,  /* 00000B68    "."......" */
+    0x00,0x01,0x09,0x00,0x00,0x09,0x3D,0x00,  /* 00000B70    "......=." */
+    0x10,0x00,0x01,0x00,0x00,0x5C,0x5F,0x53,  /* 00000B78    ".....\_S" */
+    0x42,0x5F,0x2E,0x50,0x43,0x49,0x30,0x2E,  /* 00000B80    "B_.PCI0." */
+    0x53,0x50,0x49,0x30,0x00,0x8C,0x2A,0x00,  /* 00000B88    "SPI0..*." */
+    0x01,0x01,0x01,0x00,0x08,0x00,0x00,0x00,  /* 00000B90    "........" */
+    0x00,0x00,0x00,0x17,0x00,0x00,0x19,0x00,  /* 00000B98    "........" */
+    0x2D,0x00,0x00,0x00,0x08,0x00,0x5C,0x5F,  /* 00000BA0    "-.....\_" */
+    0x53,0x42,0x2E,0x50,0x43,0x49,0x30,0x2E,  /* 00000BA8    "SB.PCI0." */
+    0x47,0x49,0x50,0x30,0x2E,0x47,0x50,0x4F,  /* 00000BB0    "GIP0.GPO" */
+    0x5F,0x00,0x79,0x00,0x14,0x0B,0x5F,0x43,  /* 00000BB8    "_.y..._C" */
+    0x52,0x53,0x00,0xA4,0x52,0x42,0x55,0x46,  /* 00000BC0    "RS..RBUF" */
+    0x14,0x15,0x5F,0x53,0x54,0x41,0x00,0xA0,  /* 00000BC8    ".._STA.." */
+    0x0B,0x92,0x93,0x50,0x54,0x59,0x50,0x0A,  /* 00000BD0    "...PTYP." */
+    0x06,0xA4,0x00,0xA4,0x0A,0x0F,0x5B,0x82,  /* 00000BD8    "......[." */
+    0x41,0x0A,0x41,0x44,0x43,0x32,0x08,0x5F,  /* 00000BE0    "A.ADC2._" */
+    0x48,0x49,0x44,0x0D,0x49,0x4E,0x54,0x33,  /* 00000BE8    "HID.INT3" */
+    0x34,0x39,0x35,0x00,0x08,0x5F,0x43,0x49,  /* 00000BF0    "495.._CI" */
+    0x44,0x0D,0x49,0x4E,0x54,0x33,0x34,0x39,  /* 00000BF8    "D.INT349" */
+    0x35,0x00,0x08,0x52,0x42,0x55,0x46,0x11,  /* 00000C00    "5..RBUF." */
+    0x47,0x05,0x0A,0x53,0x8E,0x21,0x00,0x01,  /* 00000C08    "G..S.!.." */
+    0x00,0x02,0x02,0x00,0x00,0x01,0x09,0x00,  /* 00000C10    "........" */
+    0x78,0x51,0xFE,0x00,0x10,0x01,0x01,0x00,  /* 00000C18    "xQ......" */
+    0x00,0x5C,0x5F,0x53,0x42,0x2E,0x50,0x43,  /* 00000C20    ".\_SB.PC" */
+    0x49,0x30,0x2E,0x53,0x50,0x49,0x30,0x00,  /* 00000C28    "I0.SPI0." */
+    0x8C,0x2A,0x00,0x01,0x01,0x01,0x00,0x08,  /* 00000C30    ".*......" */
+    0x00,0x00,0x00,0x00,0x00,0x00,0x17,0x00,  /* 00000C38    "........" */
+    0x00,0x19,0x00,0x2D,0x00,0x00,0x00,0x08,  /* 00000C40    "...-...." */
+    0x00,0x5C,0x5F,0x53,0x42,0x2E,0x50,0x43,  /* 00000C48    ".\_SB.PC" */
+    0x49,0x30,0x2E,0x47,0x49,0x50,0x30,0x2E,  /* 00000C50    "I0.GIP0." */
+    0x47,0x50,0x4F,0x5F,0x00,0x79,0x00,0x14,  /* 00000C58    "GPO_.y.." */
+    0x0B,0x5F,0x43,0x52,0x53,0x00,0xA4,0x52,  /* 00000C60    "._CRS..R" */
+    0x42,0x55,0x46,0x14,0x15,0x5F,0x53,0x54,  /* 00000C68    "BUF.._ST" */
+    0x41,0x00,0xA0,0x0B,0x92,0x93,0x50,0x54,  /* 00000C70    "A.....PT" */
+    0x59,0x50,0x0A,0x08,0xA4,0x00,0xA4,0x0A,  /* 00000C78    "YP......" */
+    0x0F,0x5B,0x82,0x22,0x53,0x50,0x49,0x31,  /* 00000C80    ".[."SPI1" */
+    0x08,0x5F,0x41,0x44,0x52,0x0C,0x01,0x00,  /* 00000C88    "._ADR..." */
+    0x15,0x00,0x08,0x5F,0x53,0x54,0x41,0x0A,  /* 00000C90    "..._STA." */
+    0x0F,0x08,0x5F,0x50,0x52,0x57,0x12,0x06,  /* 00000C98    ".._PRW.." */
+    0x02,0x0A,0x0F,0x0A,0x03,0x5B,0x82,0x43,  /* 00000CA0    ".....[.C" */
+    0x06,0x47,0x49,0x50,0x30,0x08,0x5F,0x41,  /* 00000CA8    ".GIP0._A" */
+    0x44,0x52,0x0C,0x02,0x00,0x15,0x00,0x08,  /* 00000CB0    "DR......" */
+    0x5F,0x53,0x54,0x41,0x0A,0x0F,0x08,0x5F,  /* 00000CB8    "_STA..._" */
+    0x50,0x52,0x57,0x12,0x06,0x02,0x0A,0x0F,  /* 00000CC0    "PRW....." */
+    0x0A,0x03,0x5B,0x82,0x1E,0x47,0x50,0x4F,  /* 00000CC8    "..[..GPO" */
+    0x5F,0x08,0x5F,0x41,0x44,0x52,0x00,0x08,  /* 00000CD0    "_._ADR.." */
+    0x5F,0x53,0x54,0x41,0x0A,0x0F,0x08,0x5F,  /* 00000CD8    "_STA..._" */
+    0x50,0x52,0x57,0x12,0x06,0x02,0x0A,0x0F,  /* 00000CE0    "PRW....." */
+    0x0A,0x03,0x5B,0x82,0x1E,0x49,0x32,0x43,  /* 00000CE8    "..[..I2C" */
+    0x5F,0x08,0x5F,0x41,0x44,0x52,0x01,0x08,  /* 00000CF0    "_._ADR.." */
+    0x5F,0x53,0x54,0x41,0x0A,0x0F,0x08,0x5F,  /* 00000CF8    "_STA..._" */
+    0x50,0x52,0x57,0x12,0x06,0x02,0x0A,0x0F,  /* 00000D00    "PRW....." */
+    0x0A,0x03,0x5B,0x82,0x4C,0x9D,0x4C,0x50,  /* 00000D08    "..[.L.LP" */
+    0x43,0x5F,0x08,0x5F,0x41,0x44,0x52,0x0C,  /* 00000D10    "C_._ADR." */
+    0x00,0x00,0x1F,0x00,0x5B,0x80,0x50,0x52,  /* 00000D18    "....[.PR" */
+    0x52,0x30,0x02,0x0A,0x60,0x0A,0x08,0x5B,  /* 00000D20    "R0..`..[" */
+    0x81,0x2E,0x50,0x52,0x52,0x30,0x00,0x50,  /* 00000D28    "..PRR0.P" */
+    0x49,0x52,0x41,0x08,0x50,0x49,0x52,0x42,  /* 00000D30    "IRA.PIRB" */
+    0x08,0x50,0x49,0x52,0x43,0x08,0x50,0x49,  /* 00000D38    ".PIRC.PI" */
+    0x52,0x44,0x08,0x50,0x49,0x52,0x45,0x08,  /* 00000D40    "RD.PIRE." */
+    0x50,0x49,0x52,0x46,0x08,0x50,0x49,0x52,  /* 00000D48    "PIRF.PIR" */
+    0x47,0x08,0x50,0x49,0x52,0x48,0x08,0x5B,  /* 00000D50    "G.PIRH.[" */
+    0x82,0x47,0x0C,0x4C,0x4E,0x4B,0x41,0x08,  /* 00000D58    ".G.LNKA." */
+    0x5F,0x48,0x49,0x44,0x0C,0x41,0xD0,0x0C,  /* 00000D60    "_HID.A.." */
+    0x0F,0x08,0x5F,0x55,0x49,0x44,0x01,0x14,  /* 00000D68    ".._UID.." */
+    0x18,0x5F,0x53,0x54,0x41,0x00,0xA0,0x0C,  /* 00000D70    "._STA..." */
+    0x7B,0x50,0x49,0x52,0x41,0x0A,0x80,0x00,  /* 00000D78    "{PIRA..." */
+    0xA4,0x0A,0x09,0xA1,0x04,0xA4,0x0A,0x0B,  /* 00000D80    "........" */
+    0x14,0x11,0x5F,0x44,0x49,0x53,0x00,0x7D,  /* 00000D88    ".._DIS.}" */
+    0x50,0x49,0x52,0x41,0x0A,0x80,0x50,0x49,  /* 00000D90    "PIRA..PI" */
+    0x52,0x41,0x14,0x45,0x04,0x5F,0x43,0x52,  /* 00000D98    "RA.E._CR" */
+    0x53,0x08,0x08,0x42,0x55,0x46,0x30,0x11,  /* 00000DA0    "S..BUF0." */
+    0x09,0x0A,0x06,0x23,0x01,0x00,0x18,0x79,  /* 00000DA8    "...#...y" */
+    0x00,0x8B,0x42,0x55,0x46,0x30,0x01,0x49,  /* 00000DB0    "..BUF0.I" */
+    0x52,0x51,0x57,0xA0,0x0C,0x7B,0x50,0x49,  /* 00000DB8    "RQW..{PI" */
+    0x52,0x41,0x0A,0x80,0x00,0x70,0x00,0x60,  /* 00000DC0    "RA...p.`" */
+    0xA1,0x04,0x70,0x01,0x60,0x79,0x60,0x7B,  /* 00000DC8    "..p.`y`{" */
+    0x50,0x49,0x52,0x41,0x0A,0x0F,0x00,0x49,  /* 00000DD0    "PIRA...I" */
+    0x52,0x51,0x57,0xA4,0x42,0x55,0x46,0x30,  /* 00000DD8    "RQW.BUF0" */
+    0x08,0x5F,0x50,0x52,0x53,0x11,0x09,0x0A,  /* 00000DE0    "._PRS..." */
+    0x06,0x23,0xB8,0x1E,0x18,0x79,0x00,0x14,  /* 00000DE8    ".#...y.." */
+    0x30,0x5F,0x53,0x52,0x53,0x01,0x8B,0x68,  /* 00000DF0    "0_SRS..h" */
+    0x01,0x49,0x52,0x51,0x57,0x82,0x49,0x52,  /* 00000DF8    ".IRQW.IR" */
+    0x51,0x57,0x60,0xA0,0x0F,0x92,0x93,0x49,  /* 00000E00    "QW`....I" */
+    0x52,0x51,0x57,0x00,0x7B,0x60,0x0A,0x7F,  /* 00000E08    "RQW.{`.." */
+    0x60,0x76,0x60,0xA1,0x06,0x7D,0x60,0x0A,  /* 00000E10    "`v`..}`." */
+    0x80,0x60,0x70,0x60,0x50,0x49,0x52,0x41,  /* 00000E18    ".`p`PIRA" */
+    0x5B,0x82,0x48,0x0C,0x4C,0x4E,0x4B,0x42,  /* 00000E20    "[.H.LNKB" */
+    0x08,0x5F,0x48,0x49,0x44,0x0C,0x41,0xD0,  /* 00000E28    "._HID.A." */
+    0x0C,0x0F,0x08,0x5F,0x55,0x49,0x44,0x0A,  /* 00000E30    "..._UID." */
+    0x02,0x14,0x18,0x5F,0x53,0x54,0x41,0x00,  /* 00000E38    "..._STA." */
+    0xA0,0x0C,0x7B,0x50,0x49,0x52,0x42,0x0A,  /* 00000E40    "..{PIRB." */
+    0x80,0x00,0xA4,0x0A,0x09,0xA1,0x04,0xA4,  /* 00000E48    "........" */
+    0x0A,0x0B,0x14,0x11,0x5F,0x44,0x49,0x53,  /* 00000E50    "...._DIS" */
+    0x00,0x7D,0x50,0x49,0x52,0x42,0x0A,0x80,  /* 00000E58    ".}PIRB.." */
+    0x50,0x49,0x52,0x42,0x14,0x45,0x04,0x5F,  /* 00000E60    "PIRB.E._" */
+    0x43,0x52,0x53,0x08,0x08,0x42,0x55,0x46,  /* 00000E68    "CRS..BUF" */
+    0x30,0x11,0x09,0x0A,0x06,0x23,0x01,0x00,  /* 00000E70    "0....#.." */
+    0x18,0x79,0x00,0x8B,0x42,0x55,0x46,0x30,  /* 00000E78    ".y..BUF0" */
+    0x01,0x49,0x52,0x51,0x57,0xA0,0x0C,0x7B,  /* 00000E80    ".IRQW..{" */
+    0x50,0x49,0x52,0x42,0x0A,0x80,0x00,0x70,  /* 00000E88    "PIRB...p" */
+    0x00,0x60,0xA1,0x04,0x70,0x01,0x60,0x79,  /* 00000E90    ".`..p.`y" */
+    0x60,0x7B,0x50,0x49,0x52,0x42,0x0A,0x0F,  /* 00000E98    "`{PIRB.." */
+    0x00,0x49,0x52,0x51,0x57,0xA4,0x42,0x55,  /* 00000EA0    ".IRQW.BU" */
+    0x46,0x30,0x08,0x5F,0x50,0x52,0x53,0x11,  /* 00000EA8    "F0._PRS." */
+    0x09,0x0A,0x06,0x23,0xB8,0x1E,0x18,0x79,  /* 00000EB0    "...#...y" */
+    0x00,0x14,0x30,0x5F,0x53,0x52,0x53,0x01,  /* 00000EB8    "..0_SRS." */
+    0x8B,0x68,0x01,0x49,0x52,0x51,0x57,0x82,  /* 00000EC0    ".h.IRQW." */
+    0x49,0x52,0x51,0x57,0x60,0xA0,0x0F,0x92,  /* 00000EC8    "IRQW`..." */
+    0x93,0x49,0x52,0x51,0x57,0x00,0x7B,0x60,  /* 00000ED0    ".IRQW.{`" */
+    0x0A,0x7F,0x60,0x76,0x60,0xA1,0x06,0x7D,  /* 00000ED8    "..`v`..}" */
+    0x60,0x0A,0x80,0x60,0x70,0x60,0x50,0x49,  /* 00000EE0    "`..`p`PI" */
+    0x52,0x42,0x5B,0x82,0x48,0x0C,0x4C,0x4E,  /* 00000EE8    "RB[.H.LN" */
+    0x4B,0x43,0x08,0x5F,0x48,0x49,0x44,0x0C,  /* 00000EF0    "KC._HID." */
+    0x41,0xD0,0x0C,0x0F,0x08,0x5F,0x55,0x49,  /* 00000EF8    "A...._UI" */
+    0x44,0x0A,0x03,0x14,0x18,0x5F,0x53,0x54,  /* 00000F00    "D...._ST" */
+    0x41,0x00,0xA0,0x0C,0x7B,0x50,0x49,0x52,  /* 00000F08    "A...{PIR" */
+    0x43,0x0A,0x80,0x00,0xA4,0x0A,0x09,0xA1,  /* 00000F10    "C......." */
+    0x04,0xA4,0x0A,0x0B,0x14,0x11,0x5F,0x44,  /* 00000F18    "......_D" */
+    0x49,0x53,0x00,0x7D,0x50,0x49,0x52,0x43,  /* 00000F20    "IS.}PIRC" */
+    0x0A,0x80,0x50,0x49,0x52,0x43,0x14,0x45,  /* 00000F28    "..PIRC.E" */
+    0x04,0x5F,0x43,0x52,0x53,0x08,0x08,0x42,  /* 00000F30    "._CRS..B" */
+    0x55,0x46,0x30,0x11,0x09,0x0A,0x06,0x23,  /* 00000F38    "UF0....#" */
+    0x01,0x00,0x18,0x79,0x00,0x8B,0x42,0x55,  /* 00000F40    "...y..BU" */
+    0x46,0x30,0x01,0x49,0x52,0x51,0x57,0xA0,  /* 00000F48    "F0.IRQW." */
+    0x0C,0x7B,0x50,0x49,0x52,0x43,0x0A,0x80,  /* 00000F50    ".{PIRC.." */
+    0x00,0x70,0x00,0x60,0xA1,0x04,0x70,0x01,  /* 00000F58    ".p.`..p." */
+    0x60,0x79,0x60,0x7B,0x50,0x49,0x52,0x43,  /* 00000F60    "`y`{PIRC" */
+    0x0A,0x0F,0x00,0x49,0x52,0x51,0x57,0xA4,  /* 00000F68    "...IRQW." */
+    0x42,0x55,0x46,0x30,0x08,0x5F,0x50,0x52,  /* 00000F70    "BUF0._PR" */
+    0x53,0x11,0x09,0x0A,0x06,0x23,0xB8,0x1E,  /* 00000F78    "S....#.." */
+    0x18,0x79,0x00,0x14,0x30,0x5F,0x53,0x52,  /* 00000F80    ".y..0_SR" */
+    0x53,0x01,0x8B,0x68,0x01,0x49,0x52,0x51,  /* 00000F88    "S..h.IRQ" */
+    0x57,0x82,0x49,0x52,0x51,0x57,0x60,0xA0,  /* 00000F90    "W.IRQW`." */
+    0x0F,0x92,0x93,0x49,0x52,0x51,0x57,0x00,  /* 00000F98    "...IRQW." */
+    0x7B,0x60,0x0A,0x7F,0x60,0x76,0x60,0xA1,  /* 00000FA0    "{`..`v`." */
+    0x06,0x7D,0x60,0x0A,0x80,0x60,0x70,0x60,  /* 00000FA8    ".}`..`p`" */
+    0x50,0x49,0x52,0x43,0x5B,0x82,0x48,0x0C,  /* 00000FB0    "PIRC[.H." */
+    0x4C,0x4E,0x4B,0x44,0x08,0x5F,0x48,0x49,  /* 00000FB8    "LNKD._HI" */
+    0x44,0x0C,0x41,0xD0,0x0C,0x0F,0x08,0x5F,  /* 00000FC0    "D.A...._" */
+    0x55,0x49,0x44,0x0A,0x04,0x14,0x18,0x5F,  /* 00000FC8    "UID...._" */
+    0x53,0x54,0x41,0x00,0xA0,0x0C,0x7B,0x50,  /* 00000FD0    "STA...{P" */
+    0x49,0x52,0x44,0x0A,0x80,0x00,0xA4,0x0A,  /* 00000FD8    "IRD....." */
+    0x09,0xA1,0x04,0xA4,0x0A,0x0B,0x14,0x11,  /* 00000FE0    "........" */
+    0x5F,0x44,0x49,0x53,0x00,0x7D,0x50,0x49,  /* 00000FE8    "_DIS.}PI" */
+    0x52,0x44,0x0A,0x80,0x50,0x49,0x52,0x44,  /* 00000FF0    "RD..PIRD" */
+    0x14,0x45,0x04,0x5F,0x43,0x52,0x53,0x08,  /* 00000FF8    ".E._CRS." */
+    0x08,0x42,0x55,0x46,0x30,0x11,0x09,0x0A,  /* 00001000    ".BUF0..." */
+    0x06,0x23,0x01,0x00,0x18,0x79,0x00,0x8B,  /* 00001008    ".#...y.." */
+    0x42,0x55,0x46,0x30,0x01,0x49,0x52,0x51,  /* 00001010    "BUF0.IRQ" */
+    0x57,0xA0,0x0C,0x7B,0x50,0x49,0x52,0x44,  /* 00001018    "W..{PIRD" */
+    0x0A,0x80,0x00,0x70,0x00,0x60,0xA1,0x04,  /* 00001020    "...p.`.." */
+    0x70,0x01,0x60,0x79,0x60,0x7B,0x50,0x49,  /* 00001028    "p.`y`{PI" */
+    0x52,0x44,0x0A,0x0F,0x00,0x49,0x52,0x51,  /* 00001030    "RD...IRQ" */
+    0x57,0xA4,0x42,0x55,0x46,0x30,0x08,0x5F,  /* 00001038    "W.BUF0._" */
+    0x50,0x52,0x53,0x11,0x09,0x0A,0x06,0x23,  /* 00001040    "PRS....#" */
+    0xB8,0x1E,0x18,0x79,0x00,0x14,0x30,0x5F,  /* 00001048    "...y..0_" */
+    0x53,0x52,0x53,0x01,0x8B,0x68,0x01,0x49,  /* 00001050    "SRS..h.I" */
+    0x52,0x51,0x57,0x82,0x49,0x52,0x51,0x57,  /* 00001058    "RQW.IRQW" */
+    0x60,0xA0,0x0F,0x92,0x93,0x49,0x52,0x51,  /* 00001060    "`....IRQ" */
+    0x57,0x00,0x7B,0x60,0x0A,0x7F,0x60,0x76,  /* 00001068    "W.{`..`v" */
+    0x60,0xA1,0x06,0x7D,0x60,0x0A,0x80,0x60,  /* 00001070    "`..}`..`" */
+    0x70,0x60,0x50,0x49,0x52,0x44,0x5B,0x82,  /* 00001078    "p`PIRD[." */
+    0x48,0x0C,0x4C,0x4E,0x4B,0x45,0x08,0x5F,  /* 00001080    "H.LNKE._" */
+    0x48,0x49,0x44,0x0C,0x41,0xD0,0x0C,0x0F,  /* 00001088    "HID.A..." */
+    0x08,0x5F,0x55,0x49,0x44,0x0A,0x05,0x14,  /* 00001090    "._UID..." */
+    0x18,0x5F,0x53,0x54,0x41,0x00,0xA0,0x0C,  /* 00001098    "._STA..." */
+    0x7B,0x50,0x49,0x52,0x45,0x0A,0x80,0x00,  /* 000010A0    "{PIRE..." */
+    0xA4,0x0A,0x09,0xA1,0x04,0xA4,0x0A,0x0B,  /* 000010A8    "........" */
+    0x14,0x11,0x5F,0x44,0x49,0x53,0x00,0x7D,  /* 000010B0    ".._DIS.}" */
+    0x50,0x49,0x52,0x45,0x0A,0x80,0x50,0x49,  /* 000010B8    "PIRE..PI" */
+    0x52,0x45,0x14,0x45,0x04,0x5F,0x43,0x52,  /* 000010C0    "RE.E._CR" */
+    0x53,0x08,0x08,0x42,0x55,0x46,0x30,0x11,  /* 000010C8    "S..BUF0." */
+    0x09,0x0A,0x06,0x23,0x01,0x00,0x18,0x79,  /* 000010D0    "...#...y" */
+    0x00,0x8B,0x42,0x55,0x46,0x30,0x01,0x49,  /* 000010D8    "..BUF0.I" */
+    0x52,0x51,0x57,0xA0,0x0C,0x7B,0x50,0x49,  /* 000010E0    "RQW..{PI" */
+    0x52,0x45,0x0A,0x80,0x00,0x70,0x00,0x60,  /* 000010E8    "RE...p.`" */
+    0xA1,0x04,0x70,0x01,0x60,0x79,0x60,0x7B,  /* 000010F0    "..p.`y`{" */
+    0x50,0x49,0x52,0x45,0x0A,0x0F,0x00,0x49,  /* 000010F8    "PIRE...I" */
+    0x52,0x51,0x57,0xA4,0x42,0x55,0x46,0x30,  /* 00001100    "RQW.BUF0" */
+    0x08,0x5F,0x50,0x52,0x53,0x11,0x09,0x0A,  /* 00001108    "._PRS..." */
+    0x06,0x23,0xB8,0x1E,0x18,0x79,0x00,0x14,  /* 00001110    ".#...y.." */
+    0x30,0x5F,0x53,0x52,0x53,0x01,0x8B,0x68,  /* 00001118    "0_SRS..h" */
+    0x01,0x49,0x52,0x51,0x57,0x82,0x49,0x52,  /* 00001120    ".IRQW.IR" */
+    0x51,0x57,0x60,0xA0,0x0F,0x92,0x93,0x49,  /* 00001128    "QW`....I" */
+    0x52,0x51,0x57,0x00,0x7B,0x60,0x0A,0x7F,  /* 00001130    "RQW.{`.." */
+    0x60,0x76,0x60,0xA1,0x06,0x7D,0x60,0x0A,  /* 00001138    "`v`..}`." */
+    0x80,0x60,0x70,0x60,0x50,0x49,0x52,0x45,  /* 00001140    ".`p`PIRE" */
+    0x5B,0x82,0x48,0x0C,0x4C,0x4E,0x4B,0x46,  /* 00001148    "[.H.LNKF" */
+    0x08,0x5F,0x48,0x49,0x44,0x0C,0x41,0xD0,  /* 00001150    "._HID.A." */
+    0x0C,0x0F,0x08,0x5F,0x55,0x49,0x44,0x0A,  /* 00001158    "..._UID." */
+    0x06,0x14,0x18,0x5F,0x53,0x54,0x41,0x00,  /* 00001160    "..._STA." */
+    0xA0,0x0C,0x7B,0x50,0x49,0x52,0x46,0x0A,  /* 00001168    "..{PIRF." */
+    0x80,0x00,0xA4,0x0A,0x09,0xA1,0x04,0xA4,  /* 00001170    "........" */
+    0x0A,0x0B,0x14,0x11,0x5F,0x44,0x49,0x53,  /* 00001178    "...._DIS" */
+    0x00,0x7D,0x50,0x49,0x52,0x42,0x0A,0x80,  /* 00001180    ".}PIRB.." */
+    0x50,0x49,0x52,0x46,0x14,0x45,0x04,0x5F,  /* 00001188    "PIRF.E._" */
+    0x43,0x52,0x53,0x08,0x08,0x42,0x55,0x46,  /* 00001190    "CRS..BUF" */
+    0x30,0x11,0x09,0x0A,0x06,0x23,0x01,0x00,  /* 00001198    "0....#.." */
+    0x18,0x79,0x00,0x8B,0x42,0x55,0x46,0x30,  /* 000011A0    ".y..BUF0" */
+    0x01,0x49,0x52,0x51,0x57,0xA0,0x0C,0x7B,  /* 000011A8    ".IRQW..{" */
+    0x50,0x49,0x52,0x46,0x0A,0x80,0x00,0x70,  /* 000011B0    "PIRF...p" */
+    0x00,0x60,0xA1,0x04,0x70,0x01,0x60,0x79,  /* 000011B8    ".`..p.`y" */
+    0x60,0x7B,0x50,0x49,0x52,0x46,0x0A,0x0F,  /* 000011C0    "`{PIRF.." */
+    0x00,0x49,0x52,0x51,0x57,0xA4,0x42,0x55,  /* 000011C8    ".IRQW.BU" */
+    0x46,0x30,0x08,0x5F,0x50,0x52,0x53,0x11,  /* 000011D0    "F0._PRS." */
+    0x09,0x0A,0x06,0x23,0xB8,0x1E,0x18,0x79,  /* 000011D8    "...#...y" */
+    0x00,0x14,0x30,0x5F,0x53,0x52,0x53,0x01,  /* 000011E0    "..0_SRS." */
+    0x8B,0x68,0x01,0x49,0x52,0x51,0x57,0x82,  /* 000011E8    ".h.IRQW." */
+    0x49,0x52,0x51,0x57,0x60,0xA0,0x0F,0x92,  /* 000011F0    "IRQW`..." */
+    0x93,0x49,0x52,0x51,0x57,0x00,0x7B,0x60,  /* 000011F8    ".IRQW.{`" */
+    0x0A,0x7F,0x60,0x76,0x60,0xA1,0x06,0x7D,  /* 00001200    "..`v`..}" */
+    0x60,0x0A,0x80,0x60,0x70,0x60,0x50,0x49,  /* 00001208    "`..`p`PI" */
+    0x52,0x46,0x5B,0x82,0x48,0x0C,0x4C,0x4E,  /* 00001210    "RF[.H.LN" */
+    0x4B,0x47,0x08,0x5F,0x48,0x49,0x44,0x0C,  /* 00001218    "KG._HID." */
+    0x41,0xD0,0x0C,0x0F,0x08,0x5F,0x55,0x49,  /* 00001220    "A...._UI" */
+    0x44,0x0A,0x07,0x14,0x18,0x5F,0x53,0x54,  /* 00001228    "D...._ST" */
+    0x41,0x00,0xA0,0x0C,0x7B,0x50,0x49,0x52,  /* 00001230    "A...{PIR" */
+    0x47,0x0A,0x80,0x00,0xA4,0x0A,0x09,0xA1,  /* 00001238    "G......." */
+    0x04,0xA4,0x0A,0x0B,0x14,0x11,0x5F,0x44,  /* 00001240    "......_D" */
+    0x49,0x53,0x00,0x7D,0x50,0x49,0x52,0x47,  /* 00001248    "IS.}PIRG" */
+    0x0A,0x80,0x50,0x49,0x52,0x47,0x14,0x45,  /* 00001250    "..PIRG.E" */
+    0x04,0x5F,0x43,0x52,0x53,0x08,0x08,0x42,  /* 00001258    "._CRS..B" */
+    0x55,0x46,0x30,0x11,0x09,0x0A,0x06,0x23,  /* 00001260    "UF0....#" */
+    0x01,0x00,0x18,0x79,0x00,0x8B,0x42,0x55,  /* 00001268    "...y..BU" */
+    0x46,0x30,0x01,0x49,0x52,0x51,0x57,0xA0,  /* 00001270    "F0.IRQW." */
+    0x0C,0x7B,0x50,0x49,0x52,0x47,0x0A,0x80,  /* 00001278    ".{PIRG.." */
+    0x00,0x70,0x00,0x60,0xA1,0x04,0x70,0x01,  /* 00001280    ".p.`..p." */
+    0x60,0x79,0x60,0x7B,0x50,0x49,0x52,0x47,  /* 00001288    "`y`{PIRG" */
+    0x0A,0x0F,0x00,0x49,0x52,0x51,0x57,0xA4,  /* 00001290    "...IRQW." */
+    0x42,0x55,0x46,0x30,0x08,0x5F,0x50,0x52,  /* 00001298    "BUF0._PR" */
+    0x53,0x11,0x09,0x0A,0x06,0x23,0xB8,0x1E,  /* 000012A0    "S....#.." */
+    0x18,0x79,0x00,0x14,0x30,0x5F,0x53,0x52,  /* 000012A8    ".y..0_SR" */
+    0x53,0x01,0x8B,0x68,0x01,0x49,0x52,0x51,  /* 000012B0    "S..h.IRQ" */
+    0x57,0x82,0x49,0x52,0x51,0x57,0x60,0xA0,  /* 000012B8    "W.IRQW`." */
+    0x0F,0x92,0x93,0x49,0x52,0x51,0x57,0x00,  /* 000012C0    "...IRQW." */
+    0x7B,0x60,0x0A,0x7F,0x60,0x76,0x60,0xA1,  /* 000012C8    "{`..`v`." */
+    0x06,0x7D,0x60,0x0A,0x80,0x60,0x70,0x60,  /* 000012D0    ".}`..`p`" */
+    0x50,0x49,0x52,0x47,0x5B,0x82,0x48,0x0C,  /* 000012D8    "PIRG[.H." */
+    0x4C,0x4E,0x4B,0x48,0x08,0x5F,0x48,0x49,  /* 000012E0    "LNKH._HI" */
+    0x44,0x0C,0x41,0xD0,0x0C,0x0F,0x08,0x5F,  /* 000012E8    "D.A...._" */
+    0x55,0x49,0x44,0x0A,0x08,0x14,0x18,0x5F,  /* 000012F0    "UID...._" */
+    0x53,0x54,0x41,0x00,0xA0,0x0C,0x7B,0x50,  /* 000012F8    "STA...{P" */
+    0x49,0x52,0x48,0x0A,0x80,0x00,0xA4,0x0A,  /* 00001300    "IRH....." */
+    0x09,0xA1,0x04,0xA4,0x0A,0x0B,0x14,0x11,  /* 00001308    "........" */
+    0x5F,0x44,0x49,0x53,0x00,0x7D,0x50,0x49,  /* 00001310    "_DIS.}PI" */
+    0x52,0x48,0x0A,0x80,0x50,0x49,0x52,0x48,  /* 00001318    "RH..PIRH" */
+    0x14,0x45,0x04,0x5F,0x43,0x52,0x53,0x08,  /* 00001320    ".E._CRS." */
+    0x08,0x42,0x55,0x46,0x30,0x11,0x09,0x0A,  /* 00001328    ".BUF0..." */
+    0x06,0x23,0x01,0x00,0x18,0x79,0x00,0x8B,  /* 00001330    ".#...y.." */
+    0x42,0x55,0x46,0x30,0x01,0x49,0x52,0x51,  /* 00001338    "BUF0.IRQ" */
+    0x57,0xA0,0x0C,0x7B,0x50,0x49,0x52,0x48,  /* 00001340    "W..{PIRH" */
+    0x0A,0x80,0x00,0x70,0x00,0x60,0xA1,0x04,  /* 00001348    "...p.`.." */
+    0x70,0x01,0x60,0x79,0x60,0x7B,0x50,0x49,  /* 00001350    "p.`y`{PI" */
+    0x52,0x48,0x0A,0x0F,0x00,0x49,0x52,0x51,  /* 00001358    "RH...IRQ" */
+    0x57,0xA4,0x42,0x55,0x46,0x30,0x08,0x5F,  /* 00001360    "W.BUF0._" */
+    0x50,0x52,0x53,0x11,0x09,0x0A,0x06,0x23,  /* 00001368    "PRS....#" */
+    0xB8,0x1E,0x18,0x79,0x00,0x14,0x30,0x5F,  /* 00001370    "...y..0_" */
+    0x53,0x52,0x53,0x01,0x8B,0x68,0x01,0x49,  /* 00001378    "SRS..h.I" */
+    0x52,0x51,0x57,0x82,0x49,0x52,0x51,0x57,  /* 00001380    "RQW.IRQW" */
+    0x60,0xA0,0x0F,0x92,0x93,0x49,0x52,0x51,  /* 00001388    "`....IRQ" */
+    0x57,0x00,0x7B,0x60,0x0A,0x7F,0x60,0x76,  /* 00001390    "W.{`..`v" */
+    0x60,0xA1,0x06,0x7D,0x60,0x0A,0x80,0x60,  /* 00001398    "`..}`..`" */
+    0x70,0x60,0x50,0x49,0x52,0x48,0x5B,0x82,  /* 000013A0    "p`PIRH[." */
+    0x42,0x06,0x52,0x54,0x43,0x5F,0x08,0x5F,  /* 000013A8    "B.RTC_._" */
+    0x48,0x49,0x44,0x0C,0x41,0xD0,0x0B,0x00,  /* 000013B0    "HID.A..." */
+    0x08,0x42,0x55,0x46,0x30,0x11,0x15,0x0A,  /* 000013B8    ".BUF0..." */
+    0x12,0x47,0x01,0x70,0x00,0x70,0x00,0x01,  /* 000013C0    ".G.p.p.." */
+    0x04,0x47,0x01,0x74,0x00,0x74,0x00,0x01,  /* 000013C8    ".G.t.t.." */
+    0x04,0x79,0x00,0x08,0x42,0x55,0x46,0x31,  /* 000013D0    ".y..BUF1" */
+    0x11,0x18,0x0A,0x15,0x47,0x01,0x70,0x00,  /* 000013D8    "....G.p." */
+    0x70,0x00,0x01,0x04,0x47,0x01,0x74,0x00,  /* 000013E0    "p...G.t." */
+    0x74,0x00,0x01,0x04,0x22,0x00,0x01,0x79,  /* 000013E8    "t..."..y" */
+    0x00,0x14,0x18,0x5F,0x43,0x52,0x53,0x08,  /* 000013F0    "..._CRS." */
+    0xA0,0x0A,0x48,0x50,0x45,0x41,0xA4,0x42,  /* 000013F8    "..HPEA.B" */
+    0x55,0x46,0x30,0xA1,0x06,0xA4,0x42,0x55,  /* 00001400    "UF0...BU" */
+    0x46,0x31,0x5B,0x82,0x30,0x50,0x49,0x43,  /* 00001408    "F1[.0PIC" */
+    0x5F,0x08,0x5F,0x48,0x49,0x44,0x0B,0x41,  /* 00001410    "_._HID.A" */
+    0xD0,0x08,0x5F,0x43,0x52,0x53,0x11,0x1D,  /* 00001418    ".._CRS.." */
+    0x0A,0x1A,0x47,0x01,0x20,0x00,0x20,0x00,  /* 00001420    "..G. . ." */
+    0x01,0x1E,0x47,0x01,0xA0,0x00,0xA0,0x00,  /* 00001428    "..G....." */
+    0x01,0x1E,0x47,0x01,0xD0,0x04,0xD0,0x04,  /* 00001430    "..G....." */
+    0x01,0x02,0x79,0x00,0x5B,0x82,0x42,0x06,  /* 00001438    "..y.[.B." */
+    0x54,0x4D,0x52,0x5F,0x08,0x5F,0x48,0x49,  /* 00001440    "TMR_._HI" */
+    0x44,0x0C,0x41,0xD0,0x01,0x00,0x08,0x42,  /* 00001448    "D.A....B" */
+    0x55,0x46,0x30,0x11,0x15,0x0A,0x12,0x47,  /* 00001450    "UF0....G" */
+    0x01,0x40,0x00,0x40,0x00,0x01,0x04,0x47,  /* 00001458    ".@.@...G" */
+    0x01,0x50,0x00,0x50,0x00,0x01,0x04,0x79,  /* 00001460    ".P.P...y" */
+    0x00,0x08,0x42,0x55,0x46,0x31,0x11,0x18,  /* 00001468    "..BUF1.." */
+    0x0A,0x15,0x47,0x01,0x40,0x00,0x40,0x00,  /* 00001470    "..G.@.@." */
+    0x01,0x04,0x47,0x01,0x50,0x00,0x50,0x00,  /* 00001478    "..G.P.P." */
+    0x01,0x04,0x22,0x01,0x00,0x79,0x00,0x14,  /* 00001480    ".."..y.." */
+    0x18,0x5F,0x43,0x52,0x53,0x08,0xA0,0x0A,  /* 00001488    "._CRS..." */
+    0x48,0x50,0x45,0x41,0xA4,0x42,0x55,0x46,  /* 00001490    "HPEA.BUF" */
+    0x30,0xA1,0x06,0xA4,0x42,0x55,0x46,0x31,  /* 00001498    "0...BUF1" */
+    0x5B,0x82,0x22,0x53,0x50,0x4B,0x52,0x08,  /* 000014A0    "[."SPKR." */
+    0x5F,0x48,0x49,0x44,0x0C,0x41,0xD0,0x08,  /* 000014A8    "_HID.A.." */
+    0x00,0x08,0x5F,0x43,0x52,0x53,0x11,0x0D,  /* 000014B0    ".._CRS.." */
+    0x0A,0x0A,0x47,0x01,0x61,0x00,0x61,0x00,  /* 000014B8    "..G.a.a." */
+    0x01,0x01,0x79,0x00,0x5B,0x82,0x49,0x19,  /* 000014C0    "..y.[.I." */
+    0x58,0x54,0x52,0x41,0x08,0x5F,0x48,0x49,  /* 000014C8    "XTRA._HI" */
+    0x44,0x0C,0x41,0xD0,0x0C,0x02,0x08,0x43,  /* 000014D0    "D.A....C" */
+    0x52,0x53,0x5F,0x11,0x46,0x08,0x0A,0x82,  /* 000014D8    "RS_.F..." */
+    0x47,0x01,0x2E,0x00,0x2E,0x00,0x01,0x02,  /* 000014E0    "G......." */
+    0x47,0x01,0x4E,0x00,0x2E,0x00,0x01,0x02,  /* 000014E8    "G.N....." */
+    0x47,0x01,0x63,0x00,0x61,0x00,0x01,0x01,  /* 000014F0    "G.c.a..." */
+    0x47,0x01,0x65,0x00,0x61,0x00,0x01,0x01,  /* 000014F8    "G.e.a..." */
+    0x47,0x01,0x67,0x00,0x61,0x00,0x01,0x01,  /* 00001500    "G.g.a..." */
+    0x47,0x01,0x80,0x00,0x80,0x00,0x01,0x01,  /* 00001508    "G......." */
+    0x47,0x01,0x84,0x00,0x84,0x00,0x01,0x04,  /* 00001510    "G......." */
+    0x47,0x01,0x88,0x00,0x88,0x00,0x01,0x01,  /* 00001518    "G......." */
+    0x47,0x01,0x8C,0x00,0x8C,0x00,0x01,0x03,  /* 00001520    "G......." */
+    0x47,0x01,0x92,0x00,0x92,0x00,0x01,0x01,  /* 00001528    "G......." */
+    0x47,0x01,0x00,0x00,0x00,0x00,0x01,0x10,  /* 00001530    "G......." */
+    0x47,0x01,0x00,0x00,0x00,0x00,0x01,0x10,  /* 00001538    "G......." */
+    0x47,0x01,0x00,0x00,0x00,0x00,0x01,0x40,  /* 00001540    "G......@" */
+    0x47,0x01,0x00,0x00,0x00,0x00,0x01,0x40,  /* 00001548    "G......@" */
+    0x47,0x01,0x00,0x00,0x00,0x00,0x01,0x40,  /* 00001550    "G......@" */
+    0x47,0x01,0x00,0x00,0x00,0x00,0x01,0x40,  /* 00001558    "G......@" */
+    0x79,0x00,0x14,0x4C,0x0F,0x5F,0x43,0x52,  /* 00001560    "y..L._CR" */
+    0x53,0x00,0x8B,0x43,0x52,0x53,0x5F,0x0A,  /* 00001568    "S..CRS_." */
+    0x52,0x4D,0x42,0x52,0x30,0x70,0x50,0x31,  /* 00001570    "RMBR0pP1" */
+    0x42,0x42,0x4D,0x42,0x52,0x30,0x8B,0x43,  /* 00001578    "BBMBR0.C" */
+    0x52,0x53,0x5F,0x0A,0x54,0x4D,0x42,0x52,  /* 00001580    "RS_.TMBR" */
+    0x31,0x70,0x50,0x31,0x42,0x42,0x4D,0x42,  /* 00001588    "1pP1BBMB" */
+    0x52,0x31,0x8B,0x43,0x52,0x53,0x5F,0x0A,  /* 00001590    "R1.CRS_." */
+    0x5A,0x4D,0x42,0x52,0x32,0x70,0x50,0x42,  /* 00001598    "ZMBR2pPB" */
+    0x41,0x42,0x4D,0x42,0x52,0x32,0x8B,0x43,  /* 000015A0    "ABMBR2.C" */
+    0x52,0x53,0x5F,0x0A,0x5C,0x4D,0x42,0x52,  /* 000015A8    "RS_.\MBR" */
+    0x33,0x70,0x50,0x42,0x41,0x42,0x4D,0x42,  /* 000015B0    "3pPBABMB" */
+    0x52,0x33,0x8B,0x43,0x52,0x53,0x5F,0x0A,  /* 000015B8    "R3.CRS_." */
+    0x62,0x4D,0x42,0x52,0x34,0x70,0x47,0x50,  /* 000015C0    "bMBR4pGP" */
+    0x30,0x42,0x4D,0x42,0x52,0x34,0x8B,0x43,  /* 000015C8    "0BMBR4.C" */
+    0x52,0x53,0x5F,0x0A,0x64,0x4D,0x42,0x52,  /* 000015D0    "RS_.dMBR" */
+    0x35,0x70,0x47,0x50,0x30,0x42,0x4D,0x42,  /* 000015D8    "5pGP0BMB" */
+    0x52,0x35,0x8B,0x43,0x52,0x53,0x5F,0x0A,  /* 000015E0    "R5.CRS_." */
+    0x6A,0x4D,0x42,0x52,0x36,0x70,0x47,0x50,  /* 000015E8    "jMBR6pGP" */
+    0x41,0x42,0x4D,0x42,0x52,0x36,0x8B,0x43,  /* 000015F0    "ABMBR6.C" */
+    0x52,0x53,0x5F,0x0A,0x6C,0x4D,0x42,0x52,  /* 000015F8    "RS_.lMBR" */
+    0x37,0x70,0x47,0x50,0x41,0x42,0x4D,0x42,  /* 00001600    "7pGPABMB" */
+    0x52,0x37,0x8B,0x43,0x52,0x53,0x5F,0x0A,  /* 00001608    "R7.CRS_." */
+    0x72,0x4D,0x42,0x52,0x38,0x70,0x53,0x4D,  /* 00001610    "rMBR8pSM" */
+    0x42,0x42,0x4D,0x42,0x52,0x38,0x8B,0x43,  /* 00001618    "BBMBR8.C" */
+    0x52,0x53,0x5F,0x0A,0x74,0x4D,0x42,0x52,  /* 00001620    "RS_.tMBR" */
+    0x39,0x70,0x53,0x4D,0x42,0x42,0x4D,0x42,  /* 00001628    "9pSMBBMB" */
+    0x52,0x39,0x8B,0x43,0x52,0x53,0x5F,0x0A,  /* 00001630    "R9.CRS_." */
+    0x7A,0x4D,0x42,0x52,0x41,0x70,0x57,0x44,  /* 00001638    "zMBRApWD" */
+    0x54,0x42,0x4D,0x42,0x52,0x41,0x8B,0x43,  /* 00001640    "TBMBRA.C" */
+    0x52,0x53,0x5F,0x0A,0x7C,0x4D,0x42,0x52,  /* 00001648    "RS_.|MBR" */
+    0x42,0x70,0x57,0x44,0x54,0x42,0x4D,0x42,  /* 00001650    "BpWDTBMB" */
+    0x52,0x42,0xA4,0x43,0x52,0x53,0x5F,0x5B,  /* 00001658    "RB.CRS_[" */
+    0x82,0x47,0x08,0x48,0x50,0x45,0x54,0x08,  /* 00001660    ".G.HPET." */
+    0x5F,0x48,0x49,0x44,0x0C,0x41,0xD0,0x01,  /* 00001668    "_HID.A.." */
+    0x03,0x08,0x42,0x55,0x46,0x30,0x11,0x17,  /* 00001670    "..BUF0.." */
+    0x0A,0x14,0x22,0x01,0x00,0x22,0x00,0x01,  /* 00001678    ".."..".." */
+    0x86,0x09,0x00,0x00,0x00,0x00,0x00,0x00,  /* 00001680    "........" */
+    0x00,0x00,0x00,0x00,0x79,0x00,0x14,0x25,  /* 00001688    "....y..%" */
+    0x5F,0x53,0x54,0x41,0x00,0xA0,0x11,0x92,  /* 00001690    "_STA...." */
+    0x95,0x4F,0x53,0x54,0x50,0x01,0xA0,0x08,  /* 00001698    ".OSTP..." */
+    0x48,0x50,0x45,0x41,0xA4,0x0A,0x0F,0xA1,  /* 000016A0    "HPEA...." */
+    0x0A,0xA0,0x08,0x48,0x50,0x45,0x41,0xA4,  /* 000016A8    "...HPEA." */
+    0x0A,0x0B,0xA4,0x00,0x14,0x33,0x5F,0x43,  /* 000016B0    ".....3_C" */
+    0x52,0x53,0x08,0x8A,0x42,0x55,0x46,0x30,  /* 000016B8    "RS..BUF0" */
+    0x0A,0x0A,0x4D,0x42,0x52,0x30,0x70,0x48,  /* 000016C0    "..MBR0pH" */
+    0x50,0x54,0x42,0x4D,0x42,0x52,0x30,0x8A,  /* 000016C8    "PTBMBR0." */
+    0x42,0x55,0x46,0x30,0x0A,0x0E,0x4D,0x42,  /* 000016D0    "BUF0..MB" */
+    0x52,0x31,0x70,0x48,0x50,0x54,0x53,0x4D,  /* 000016D8    "R1pHPTSM" */
+    0x42,0x52,0x31,0xA4,0x42,0x55,0x46,0x30,  /* 000016E0    "BR1.BUF0" */
+    0x5B,0x82,0x4B,0x05,0x41,0x50,0x49,0x43,  /* 000016E8    "[.K.APIC" */
+    0x08,0x5F,0x48,0x49,0x44,0x0C,0x41,0xD0,  /* 000016F0    "._HID.A." */
+    0x00,0x03,0x08,0x43,0x52,0x53,0x5F,0x11,  /* 000016F8    "...CRS_." */
+    0x11,0x0A,0x0E,0x86,0x09,0x00,0x00,0x00,  /* 00001700    "........" */
+    0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x79,  /* 00001708    ".......y" */
+    0x00,0x14,0x33,0x5F,0x43,0x52,0x53,0x00,  /* 00001710    "..3_CRS." */
+    0x8A,0x43,0x52,0x53,0x5F,0x0A,0x04,0x4D,  /* 00001718    ".CRS_..M" */
+    0x42,0x52,0x30,0x70,0x41,0x50,0x43,0x42,  /* 00001720    "BR0pAPCB" */
+    0x4D,0x42,0x52,0x30,0x8A,0x43,0x52,0x53,  /* 00001728    "MBR0.CRS" */
+    0x5F,0x0A,0x08,0x4D,0x42,0x52,0x31,0x70,  /* 00001730    "_..MBR1p" */
+    0x41,0x50,0x43,0x53,0x4D,0x42,0x52,0x31,  /* 00001738    "APCSMBR1" */
+    0xA4,0x43,0x52,0x53,0x5F,0x5B,0x82,0x47,  /* 00001740    ".CRS_[.G" */
+    0x0B,0x54,0x50,0x4D,0x5F,0x08,0x5F,0x48,  /* 00001748    ".TPM_._H" */
+    0x49,0x44,0x0C,0x25,0xD4,0x34,0x93,0x08,  /* 00001750    "ID.%.4.." */
+    0x5F,0x43,0x49,0x44,0x0C,0x41,0xD0,0x0C,  /* 00001758    "_CID.A.." */
+    0x31,0x08,0x5F,0x53,0x54,0x52,0x11,0x42,  /* 00001760    "1._STR.B" */
+    0x05,0x0A,0x4E,0x49,0x00,0x6E,0x00,0x66,  /* 00001768    "..NI.n.f" */
+    0x00,0x69,0x00,0x6E,0x00,0x65,0x00,0x6F,  /* 00001770    ".i.n.e.o" */
+    0x00,0x6E,0x00,0x20,0x00,0x54,0x00,0x50,  /* 00001778    ".n. .T.P" */
+    0x00,0x4D,0x00,0x20,0x00,0x31,0x00,0x2E,  /* 00001780    ".M. .1.." */
+    0x00,0x32,0x00,0x20,0x00,0x44,0x00,0x65,  /* 00001788    ".2. .D.e" */
+    0x00,0x76,0x00,0x69,0x00,0x63,0x00,0x65,  /* 00001790    ".v.i.c.e" */
+    0x00,0x20,0x00,0x28,0x00,0x53,0x00,0x4C,  /* 00001798    ". .(.S.L" */
+    0x00,0x42,0x00,0x39,0x00,0x36,0x00,0x34,  /* 000017A0    ".B.9.6.4" */
+    0x00,0x35,0x00,0x54,0x00,0x54,0x00,0x31,  /* 000017A8    ".5.T.T.1" */
+    0x00,0x2E,0x00,0x32,0x00,0x29,0x00,0x00,  /* 000017B0    "...2.).." */
+    0x00,0x08,0x5F,0x43,0x52,0x53,0x11,0x2B,  /* 000017B8    ".._CRS.+" */
+    0x0A,0x28,0x8E,0x23,0x00,0x01,0x00,0x01,  /* 000017C0    ".(.#...." */
+    0x02,0x00,0x00,0x01,0x06,0x00,0x80,0x1A,  /* 000017C8    "........" */
+    0x06,0x00,0x20,0x00,0x5C,0x5F,0x53,0x42,  /* 000017D0    ".. .\_SB" */
+    0x2E,0x50,0x43,0x49,0x30,0x2E,0x47,0x49,  /* 000017D8    ".PCI0.GI" */
+    0x50,0x30,0x2E,0x49,0x32,0x43,0x5F,0x00,  /* 000017E0    "P0.I2C_." */
+    0x79,0x00,0x14,0x13,0x5F,0x53,0x54,0x41,  /* 000017E8    "y..._STA" */
+    0x00,0xA0,0x09,0x93,0x54,0x50,0x4D,0x50,  /* 000017F0    "....TPMP" */
+    0x00,0xA4,0x00,0xA4,0x0A,0x0F,0x5B,0x82,  /* 000017F8    "......[." */
+    0x45,0x0C,0x43,0x59,0x38,0x43,0x08,0x5F,  /* 00001800    "E.CY8C._" */
+    0x48,0x49,0x44,0x0D,0x49,0x4E,0x54,0x33,  /* 00001808    "HID.INT3" */
+    0x34,0x39,0x30,0x00,0x08,0x5F,0x43,0x49,  /* 00001810    "490.._CI" */
+    0x44,0x0D,0x49,0x4E,0x54,0x33,0x34,0x39,  /* 00001818    "D.INT349" */
+    0x30,0x00,0x08,0x52,0x42,0x55,0x46,0x11,  /* 00001820    "0..RBUF." */
+    0x48,0x05,0x0A,0x54,0x8E,0x23,0x00,0x01,  /* 00001828    "H..T.#.." */
+    0x00,0x01,0x02,0x00,0x00,0x01,0x06,0x00,  /* 00001830    "........" */
+    0xA0,0x86,0x01,0x00,0x20,0x00,0x5C,0x5F,  /* 00001838    ".... .\_" */
+    0x53,0x42,0x2E,0x50,0x43,0x49,0x30,0x2E,  /* 00001840    "SB.PCI0." */
+    0x47,0x49,0x50,0x30,0x2E,0x49,0x32,0x43,  /* 00001848    "GIP0.I2C" */
+    0x5F,0x00,0x8C,0x29,0x00,0x01,0x00,0x01,  /* 00001850    "_..)...." */
+    0x00,0x02,0x00,0x00,0x00,0x00,0x00,0x00,  /* 00001858    "........" */
+    0x17,0x00,0x00,0x19,0x00,0x2C,0x00,0x00,  /* 00001860    ".....,.." */
+    0x00,0x0D,0x00,0x5C,0x5F,0x53,0x42,0x2E,  /* 00001868    "...\_SB." */
+    0x50,0x43,0x49,0x30,0x2E,0x47,0x49,0x50,  /* 00001870    "PCI0.GIP" */
+    0x30,0x2E,0x47,0x50,0x4F,0x00,0x79,0x00,  /* 00001878    "0.GPO.y." */
+    0x14,0x2E,0x5F,0x43,0x52,0x53,0x00,0x8C,  /* 00001880    ".._CRS.." */
+    0x52,0x42,0x55,0x46,0x0A,0x10,0x4F,0x42,  /* 00001888    "RBUF..OB" */
+    0x31,0x5F,0xA0,0x0E,0x93,0x41,0x4C,0x54,  /* 00001890    "1_...ALT" */
+    0x53,0x00,0x70,0x0A,0x20,0x4F,0x42,0x31,  /* 00001898    "S.p. OB1" */
+    0x5F,0xA1,0x08,0x70,0x0A,0x21,0x4F,0x42,  /* 000018A0    "_..p.!OB" */
+    0x31,0x5F,0xA4,0x52,0x42,0x55,0x46,0x14,  /* 000018A8    "1_.RBUF." */
+    0x15,0x5F,0x53,0x54,0x41,0x00,0xA0,0x0B,  /* 000018B0    "._STA..." */
+    0x92,0x93,0x50,0x54,0x59,0x50,0x0A,0x06,  /* 000018B8    "..PTYP.." */
+    0xA4,0x00,0xA4,0x0A,0x0F,0x5B,0x82,0x4B,  /* 000018C0    ".....[.K" */
+    0x07,0x4E,0x49,0x4F,0x31,0x08,0x5F,0x48,  /* 000018C8    ".NIO1._H" */
+    0x49,0x44,0x0D,0x49,0x4E,0x54,0x33,0x34,  /* 000018D0    "ID.INT34" */
+    0x39,0x31,0x00,0x08,0x5F,0x43,0x49,0x44,  /* 000018D8    "91.._CID" */
+    0x0D,0x49,0x4E,0x54,0x33,0x34,0x39,0x31,  /* 000018E0    ".INT3491" */
+    0x00,0x08,0x5F,0x55,0x49,0x44,0x01,0x08,  /* 000018E8    ".._UID.." */
+    0x52,0x42,0x55,0x46,0x11,0x2B,0x0A,0x28,  /* 000018F0    "RBUF.+.(" */
+    0x8E,0x23,0x00,0x01,0x00,0x01,0x02,0x00,  /* 000018F8    ".#......" */
+    0x00,0x01,0x06,0x00,0x80,0x1A,0x06,0x00,  /* 00001900    "........" */
+    0x25,0x00,0x5C,0x5F,0x53,0x42,0x2E,0x50,  /* 00001908    "%.\_SB.P" */
+    0x43,0x49,0x30,0x2E,0x47,0x49,0x50,0x30,  /* 00001910    "CI0.GIP0" */
+    0x2E,0x49,0x32,0x43,0x5F,0x00,0x79,0x00,  /* 00001918    ".I2C_.y." */
+    0x14,0x0B,0x5F,0x43,0x52,0x53,0x00,0xA4,  /* 00001920    ".._CRS.." */
+    0x52,0x42,0x55,0x46,0x14,0x15,0x5F,0x53,  /* 00001928    "RBUF.._S" */
+    0x54,0x41,0x00,0xA0,0x0B,0x92,0x93,0x50,  /* 00001930    "TA.....P" */
+    0x54,0x59,0x50,0x0A,0x08,0xA4,0x00,0xA4,  /* 00001938    "TYP....." */
+    0x0A,0x0F,0x5B,0x82,0x4C,0x07,0x4E,0x49,  /* 00001940    "..[.L.NI" */
+    0x4F,0x32,0x08,0x5F,0x48,0x49,0x44,0x0D,  /* 00001948    "O2._HID." */
+    0x49,0x4E,0x54,0x33,0x34,0x39,0x31,0x00,  /* 00001950    "INT3491." */
+    0x08,0x5F,0x43,0x49,0x44,0x0D,0x49,0x4E,  /* 00001958    "._CID.IN" */
+    0x54,0x33,0x34,0x39,0x31,0x00,0x08,0x5F,  /* 00001960    "T3491.._" */
+    0x55,0x49,0x44,0x0A,0x02,0x08,0x52,0x42,  /* 00001968    "UID...RB" */
+    0x55,0x46,0x11,0x2B,0x0A,0x28,0x8E,0x23,  /* 00001970    "UF.+.(.#" */
+    0x00,0x01,0x00,0x01,0x02,0x00,0x00,0x01,  /* 00001978    "........" */
+    0x06,0x00,0x80,0x1A,0x06,0x00,0x26,0x00,  /* 00001980    "......&." */
+    0x5C,0x5F,0x53,0x42,0x2E,0x50,0x43,0x49,  /* 00001988    "\_SB.PCI" */
+    0x30,0x2E,0x47,0x49,0x50,0x30,0x2E,0x49,  /* 00001990    "0.GIP0.I" */
+    0x32,0x43,0x5F,0x00,0x79,0x00,0x14,0x0B,  /* 00001998    "2C_.y..." */
+    0x5F,0x43,0x52,0x53,0x00,0xA4,0x52,0x42,  /* 000019A0    "_CRS..RB" */
+    0x55,0x46,0x14,0x15,0x5F,0x53,0x54,0x41,  /* 000019A8    "UF.._STA" */
+    0x00,0xA0,0x0B,0x92,0x93,0x50,0x54,0x59,  /* 000019B0    ".....PTY" */
+    0x50,0x0A,0x08,0xA4,0x00,0xA4,0x0A,0x0F,  /* 000019B8    "P......." */
+    0x5B,0x82,0x49,0x0A,0x4E,0x49,0x4F,0x33,  /* 000019C0    "[.I.NIO3" */
+    0x08,0x5F,0x48,0x49,0x44,0x0D,0x49,0x4E,  /* 000019C8    "._HID.IN" */
+    0x54,0x33,0x34,0x39,0x31,0x00,0x08,0x5F,  /* 000019D0    "T3491.._" */
+    0x43,0x49,0x44,0x0D,0x49,0x4E,0x54,0x33,  /* 000019D8    "CID.INT3" */
+    0x34,0x39,0x31,0x00,0x08,0x5F,0x55,0x49,  /* 000019E0    "491.._UI" */
+    0x44,0x0A,0x03,0x08,0x52,0x42,0x55,0x46,  /* 000019E8    "D...RBUF" */
+    0x11,0x48,0x05,0x0A,0x54,0x8E,0x23,0x00,  /* 000019F0    ".H..T.#." */
+    0x01,0x00,0x01,0x02,0x00,0x00,0x01,0x06,  /* 000019F8    "........" */
+    0x00,0x80,0x1A,0x06,0x00,0x27,0x00,0x5C,  /* 00001A00    ".....'.\" */
+    0x5F,0x53,0x42,0x2E,0x50,0x43,0x49,0x30,  /* 00001A08    "_SB.PCI0" */
+    0x2E,0x47,0x49,0x50,0x30,0x2E,0x49,0x32,  /* 00001A10    ".GIP0.I2" */
+    0x43,0x5F,0x00,0x8C,0x29,0x00,0x01,0x00,  /* 00001A18    "C_..)..." */
+    0x01,0x00,0x02,0x00,0x00,0x00,0x00,0x00,  /* 00001A20    "........" */
+    0x00,0x17,0x00,0x00,0x19,0x00,0x2C,0x00,  /* 00001A28    "......,." */
+    0x00,0x00,0x01,0x00,0x5C,0x5F,0x53,0x42,  /* 00001A30    "....\_SB" */
+    0x2E,0x50,0x43,0x49,0x30,0x2E,0x47,0x49,  /* 00001A38    ".PCI0.GI" */
+    0x50,0x30,0x2E,0x47,0x50,0x4F,0x00,0x79,  /* 00001A40    "P0.GPO.y" */
+    0x00,0x14,0x0B,0x5F,0x43,0x52,0x53,0x00,  /* 00001A48    "..._CRS." */
+    0xA4,0x52,0x42,0x55,0x46,0x14,0x15,0x5F,  /* 00001A50    ".RBUF.._" */
+    0x53,0x54,0x41,0x00,0xA0,0x0B,0x92,0x93,  /* 00001A58    "STA....." */
+    0x50,0x54,0x59,0x50,0x0A,0x08,0xA4,0x00,  /* 00001A60    "PTYP...." */
+    0xA4,0x0A,0x0F,0x5B,0x82,0x45,0x07,0x50,  /* 00001A68    "...[.E.P" */
+    0x57,0x4D,0x31,0x08,0x5F,0x48,0x49,0x44,  /* 00001A70    "WM1._HID" */
+    0x0D,0x49,0x4E,0x54,0x33,0x34,0x39,0x32,  /* 00001A78    ".INT3492" */
+    0x00,0x08,0x5F,0x43,0x49,0x44,0x0D,0x49,  /* 00001A80    ".._CID.I" */
+    0x4E,0x54,0x33,0x34,0x39,0x32,0x00,0x08,  /* 00001A88    "NT3492.." */
+    0x52,0x42,0x55,0x46,0x11,0x2B,0x0A,0x28,  /* 00001A90    "RBUF.+.(" */
+    0x8E,0x23,0x00,0x01,0x00,0x01,0x02,0x00,  /* 00001A98    ".#......" */
+    0x00,0x01,0x06,0x00,0x80,0x1A,0x06,0x00,  /* 00001AA0    "........" */
+    0x47,0x00,0x5C,0x5F,0x53,0x42,0x2E,0x50,  /* 00001AA8    "G.\_SB.P" */
+    0x43,0x49,0x30,0x2E,0x47,0x49,0x50,0x30,  /* 00001AB0    "CI0.GIP0" */
+    0x2E,0x49,0x32,0x43,0x5F,0x00,0x79,0x00,  /* 00001AB8    ".I2C_.y." */
+    0x14,0x0B,0x5F,0x43,0x52,0x53,0x00,0xA4,  /* 00001AC0    ".._CRS.." */
+    0x52,0x42,0x55,0x46,0x14,0x15,0x5F,0x53,  /* 00001AC8    "RBUF.._S" */
+    0x54,0x41,0x00,0xA0,0x0B,0x92,0x93,0x50,  /* 00001AD0    "TA.....P" */
+    0x54,0x59,0x50,0x0A,0x08,0xA4,0x00,0xA4,  /* 00001AD8    "TYP....." */
+    0x0A,0x0F,0x5B,0x82,0x45,0x07,0x45,0x45,  /* 00001AE0    "..[.E.EE" */
+    0x50,0x32,0x08,0x5F,0x48,0x49,0x44,0x0D,  /* 00001AE8    "P2._HID." */
+    0x49,0x4E,0x54,0x33,0x34,0x39,0x39,0x00,  /* 00001AF0    "INT3499." */
+    0x08,0x5F,0x43,0x49,0x44,0x0D,0x49,0x4E,  /* 00001AF8    "._CID.IN" */
+    0x54,0x33,0x34,0x39,0x39,0x00,0x08,0x52,  /* 00001B00    "T3499..R" */
+    0x42,0x55,0x46,0x11,0x2B,0x0A,0x28,0x8E,  /* 00001B08    "BUF.+.(." */
+    0x23,0x00,0x01,0x00,0x01,0x02,0x00,0x00,  /* 00001B10    "#......." */
+    0x01,0x06,0x00,0x80,0x1A,0x06,0x00,0x54,  /* 00001B18    ".......T" */
+    0x00,0x5C,0x5F,0x53,0x42,0x2E,0x50,0x43,  /* 00001B20    ".\_SB.PC" */
+    0x49,0x30,0x2E,0x47,0x49,0x50,0x30,0x2E,  /* 00001B28    "I0.GIP0." */
+    0x49,0x32,0x43,0x5F,0x00,0x79,0x00,0x14,  /* 00001B30    "I2C_.y.." */
+    0x0B,0x5F,0x43,0x52,0x53,0x00,0xA4,0x52,  /* 00001B38    "._CRS..R" */
+    0x42,0x55,0x46,0x14,0x15,0x5F,0x53,0x54,  /* 00001B40    "BUF.._ST" */
+    0x41,0x00,0xA0,0x0B,0x92,0x93,0x50,0x54,  /* 00001B48    "A.....PT" */
+    0x59,0x50,0x0A,0x08,0xA4,0x00,0xA4,0x0A,  /* 00001B50    "YP......" */
+    0x0F,0x5B,0x82,0x4A,0x89,0x47,0x50,0x4F,  /* 00001B58    ".[.J.GPO" */
+    0x54,0x08,0x5F,0x48,0x49,0x44,0x0D,0x49,  /* 00001B60    "T._HID.I" */
+    0x4E,0x54,0x33,0x34,0x39,0x41,0x00,0x08,  /* 00001B68    "NT349A.." */
+    0x5F,0x43,0x49,0x44,0x0D,0x49,0x4E,0x54,  /* 00001B70    "_CID.INT" */
+    0x33,0x34,0x39,0x41,0x00,0x14,0x41,0x86,  /* 00001B78    "349A..A." */
+    0x5F,0x43,0x52,0x53,0x00,0x08,0x52,0x42,  /* 00001B80    "_CRS..RB" */
+    0x55,0x46,0x11,0x4F,0x84,0x0B,0x4A,0x08,  /* 00001B88    "UF.O..J." */
+    0x8C,0x2A,0x00,0x01,0x01,0x01,0x00,0x08,  /* 00001B90    ".*......" */
+    0x00,0x00,0x00,0x00,0x00,0x00,0x17,0x00,  /* 00001B98    "........" */
+    0x00,0x19,0x00,0x2D,0x00,0x00,0x00,0x00,  /* 00001BA0    "...-...." */
+    0x00,0x5C,0x5F,0x53,0x42,0x2E,0x50,0x43,  /* 00001BA8    ".\_SB.PC" */
+    0x49,0x30,0x2E,0x47,0x49,0x50,0x30,0x2E,  /* 00001BB0    "I0.GIP0." */
+    0x47,0x50,0x4F,0x5F,0x00,0x8C,0x2A,0x00,  /* 00001BB8    "GPO_..*." */
+    0x01,0x01,0x01,0x00,0x08,0x00,0x00,0x00,  /* 00001BC0    "........" */
+    0x00,0x00,0x00,0x17,0x00,0x00,0x19,0x00,  /* 00001BC8    "........" */
+    0x2D,0x00,0x00,0x00,0x01,0x00,0x5C,0x5F,  /* 00001BD0    "-.....\_" */
+    0x53,0x42,0x2E,0x50,0x43,0x49,0x30,0x2E,  /* 00001BD8    "SB.PCI0." */
+    0x47,0x49,0x50,0x30,0x2E,0x47,0x50,0x4F,  /* 00001BE0    "GIP0.GPO" */
+    0x5F,0x00,0x8C,0x2A,0x00,0x01,0x01,0x01,  /* 00001BE8    "_..*...." */
+    0x00,0x08,0x00,0x00,0x00,0x00,0x00,0x00,  /* 00001BF0    "........" */
+    0x17,0x00,0x00,0x19,0x00,0x2D,0x00,0x00,  /* 00001BF8    ".....-.." */
+    0x00,0x02,0x00,0x5C,0x5F,0x53,0x42,0x2E,  /* 00001C00    "...\_SB." */
+    0x50,0x43,0x49,0x30,0x2E,0x47,0x49,0x50,  /* 00001C08    "PCI0.GIP" */
+    0x30,0x2E,0x47,0x50,0x4F,0x5F,0x00,0x8C,  /* 00001C10    "0.GPO_.." */
+    0x2A,0x00,0x01,0x01,0x01,0x00,0x08,0x00,  /* 00001C18    "*......." */
+    0x00,0x00,0x00,0x00,0x00,0x17,0x00,0x00,  /* 00001C20    "........" */
+    0x19,0x00,0x2D,0x00,0x00,0x00,0x03,0x00,  /* 00001C28    "..-....." */
+    0x5C,0x5F,0x53,0x42,0x2E,0x50,0x43,0x49,  /* 00001C30    "\_SB.PCI" */
+    0x30,0x2E,0x47,0x49,0x50,0x30,0x2E,0x47,  /* 00001C38    "0.GIP0.G" */
+    0x50,0x4F,0x5F,0x00,0x8C,0x2A,0x00,0x01,  /* 00001C40    "PO_..*.." */
+    0x01,0x01,0x00,0x08,0x00,0x00,0x00,0x00,  /* 00001C48    "........" */
+    0x00,0x00,0x17,0x00,0x00,0x19,0x00,0x2D,  /* 00001C50    ".......-" */
+    0x00,0x00,0x00,0x04,0x00,0x5C,0x5F,0x53,  /* 00001C58    ".....\_S" */
+    0x42,0x2E,0x50,0x43,0x49,0x30,0x2E,0x47,  /* 00001C60    "B.PCI0.G" */
+    0x49,0x50,0x30,0x2E,0x47,0x50,0x4F,0x5F,  /* 00001C68    "IP0.GPO_" */
+    0x00,0x8C,0x2A,0x00,0x01,0x01,0x01,0x00,  /* 00001C70    "..*....." */
+    0x08,0x00,0x00,0x00,0x00,0x00,0x00,0x17,  /* 00001C78    "........" */
+    0x00,0x00,0x19,0x00,0x2D,0x00,0x00,0x00,  /* 00001C80    "....-..." */
+    0x05,0x00,0x5C,0x5F,0x53,0x42,0x2E,0x50,  /* 00001C88    "..\_SB.P" */
+    0x43,0x49,0x30,0x2E,0x47,0x49,0x50,0x30,  /* 00001C90    "CI0.GIP0" */
+    0x2E,0x47,0x50,0x4F,0x5F,0x00,0x8C,0x2A,  /* 00001C98    ".GPO_..*" */
+    0x00,0x01,0x01,0x01,0x00,0x08,0x00,0x00,  /* 00001CA0    "........" */
+    0x00,0x00,0x00,0x00,0x17,0x00,0x00,0x19,  /* 00001CA8    "........" */
+    0x00,0x2D,0x00,0x00,0x00,0x06,0x00,0x5C,  /* 00001CB0    ".-.....\" */
+    0x5F,0x53,0x42,0x2E,0x50,0x43,0x49,0x30,  /* 00001CB8    "_SB.PCI0" */
+    0x2E,0x47,0x49,0x50,0x30,0x2E,0x47,0x50,  /* 00001CC0    ".GIP0.GP" */
+    0x4F,0x5F,0x00,0x8C,0x2A,0x00,0x01,0x01,  /* 00001CC8    "O_..*..." */
+    0x01,0x00,0x08,0x00,0x00,0x00,0x00,0x00,  /* 00001CD0    "........" */
+    0x00,0x17,0x00,0x00,0x19,0x00,0x2D,0x00,  /* 00001CD8    "......-." */
+    0x00,0x00,0x07,0x00,0x5C,0x5F,0x53,0x42,  /* 00001CE0    "....\_SB" */
+    0x2E,0x50,0x43,0x49,0x30,0x2E,0x47,0x49,  /* 00001CE8    ".PCI0.GI" */
+    0x50,0x30,0x2E,0x47,0x50,0x4F,0x5F,0x00,  /* 00001CF0    "P0.GPO_." */
+    0x8C,0x2A,0x00,0x01,0x01,0x01,0x00,0x08,  /* 00001CF8    ".*......" */
+    0x00,0x00,0x00,0x00,0x00,0x00,0x17,0x00,  /* 00001D00    "........" */
+    0x00,0x19,0x00,0x2D,0x00,0x00,0x00,0x08,  /* 00001D08    "...-...." */
+    0x00,0x5C,0x5F,0x53,0x42,0x2E,0x50,0x43,  /* 00001D10    ".\_SB.PC" */
+    0x49,0x30,0x2E,0x47,0x49,0x50,0x30,0x2E,  /* 00001D18    "I0.GIP0." */
+    0x47,0x50,0x4F,0x5F,0x00,0x8C,0x2A,0x00,  /* 00001D20    "GPO_..*." */
+    0x01,0x01,0x01,0x00,0x08,0x00,0x00,0x00,  /* 00001D28    "........" */
+    0x00,0x00,0x00,0x17,0x00,0x00,0x19,0x00,  /* 00001D30    "........" */
+    0x2D,0x00,0x00,0x00,0x09,0x00,0x5C,0x5F,  /* 00001D38    "-.....\_" */
+    0x53,0x42,0x2E,0x50,0x43,0x49,0x30,0x2E,  /* 00001D40    "SB.PCI0." */
+    0x47,0x49,0x50,0x30,0x2E,0x47,0x50,0x4F,  /* 00001D48    "GIP0.GPO" */
+    0x5F,0x00,0x8C,0x2A,0x00,0x01,0x01,0x01,  /* 00001D50    "_..*...." */
+    0x00,0x08,0x00,0x00,0x00,0x00,0x00,0x00,  /* 00001D58    "........" */
+    0x17,0x00,0x00,0x19,0x00,0x2D,0x00,0x00,  /* 00001D60    ".....-.." */
+    0x00,0x0A,0x00,0x5C,0x5F,0x53,0x42,0x2E,  /* 00001D68    "...\_SB." */
+    0x50,0x43,0x49,0x30,0x2E,0x47,0x49,0x50,  /* 00001D70    "PCI0.GIP" */
+    0x30,0x2E,0x47,0x50,0x4F,0x5F,0x00,0x8C,  /* 00001D78    "0.GPO_.." */
+    0x2A,0x00,0x01,0x01,0x01,0x00,0x08,0x00,  /* 00001D80    "*......." */
+    0x00,0x00,0x00,0x00,0x00,0x17,0x00,0x00,  /* 00001D88    "........" */
+    0x19,0x00,0x2D,0x00,0x00,0x00,0x0B,0x00,  /* 00001D90    "..-....." */
+    0x5C,0x5F,0x53,0x42,0x2E,0x50,0x43,0x49,  /* 00001D98    "\_SB.PCI" */
+    0x30,0x2E,0x47,0x49,0x50,0x30,0x2E,0x47,  /* 00001DA0    "0.GIP0.G" */
+    0x50,0x4F,0x5F,0x00,0x8C,0x2A,0x00,0x01,  /* 00001DA8    "PO_..*.." */
+    0x01,0x01,0x00,0x08,0x00,0x00,0x00,0x00,  /* 00001DB0    "........" */
+    0x00,0x00,0x17,0x00,0x00,0x19,0x00,0x2D,  /* 00001DB8    ".......-" */
+    0x00,0x00,0x00,0x0C,0x00,0x5C,0x5F,0x53,  /* 00001DC0    ".....\_S" */
+    0x42,0x2E,0x50,0x43,0x49,0x30,0x2E,0x47,  /* 00001DC8    "B.PCI0.G" */
+    0x49,0x50,0x30,0x2E,0x47,0x50,0x4F,0x5F,  /* 00001DD0    "IP0.GPO_" */
+    0x00,0x8C,0x2A,0x00,0x01,0x01,0x01,0x00,  /* 00001DD8    "..*....." */
+    0x08,0x00,0x00,0x00,0x00,0x00,0x00,0x17,  /* 00001DE0    "........" */
+    0x00,0x00,0x19,0x00,0x2D,0x00,0x00,0x00,  /* 00001DE8    "....-..." */
+    0x0D,0x00,0x5C,0x5F,0x53,0x42,0x2E,0x50,  /* 00001DF0    "..\_SB.P" */
+    0x43,0x49,0x30,0x2E,0x47,0x49,0x50,0x30,  /* 00001DF8    "CI0.GIP0" */
+    0x2E,0x47,0x50,0x4F,0x5F,0x00,0x8C,0x2A,  /* 00001E00    ".GPO_..*" */
+    0x00,0x01,0x01,0x01,0x00,0x08,0x00,0x00,  /* 00001E08    "........" */
+    0x00,0x00,0x00,0x00,0x17,0x00,0x00,0x19,  /* 00001E10    "........" */
+    0x00,0x2D,0x00,0x00,0x00,0x0E,0x00,0x5C,  /* 00001E18    ".-.....\" */
+    0x5F,0x53,0x42,0x2E,0x50,0x43,0x49,0x30,  /* 00001E20    "_SB.PCI0" */
+    0x2E,0x47,0x49,0x50,0x30,0x2E,0x47,0x50,  /* 00001E28    ".GIP0.GP" */
+    0x4F,0x5F,0x00,0x8C,0x2A,0x00,0x01,0x01,  /* 00001E30    "O_..*..." */
+    0x01,0x00,0x08,0x00,0x00,0x00,0x00,0x00,  /* 00001E38    "........" */
+    0x00,0x17,0x00,0x00,0x19,0x00,0x2D,0x00,  /* 00001E40    "......-." */
+    0x00,0x00,0x0F,0x00,0x5C,0x5F,0x53,0x42,  /* 00001E48    "....\_SB" */
+    0x2E,0x50,0x43,0x49,0x30,0x2E,0x47,0x49,  /* 00001E50    ".PCI0.GI" */
+    0x50,0x30,0x2E,0x47,0x50,0x4F,0x5F,0x00,  /* 00001E58    "P0.GPO_." */
+    0x8C,0x20,0x00,0x01,0x01,0x01,0x00,0x08,  /* 00001E60    ". ......" */
+    0x00,0x00,0x00,0x00,0x00,0x00,0x17,0x00,  /* 00001E68    "........" */
+    0x00,0x19,0x00,0x23,0x00,0x00,0x00,0x00,  /* 00001E70    "...#...." */
+    0x00,0x5C,0x5F,0x53,0x42,0x2E,0x43,0x59,  /* 00001E78    ".\_SB.CY" */
+    0x38,0x43,0x00,0x8C,0x20,0x00,0x01,0x01,  /* 00001E80    "8C.. ..." */
+    0x01,0x00,0x08,0x00,0x00,0x00,0x00,0x00,  /* 00001E88    "........" */
+    0x00,0x17,0x00,0x00,0x19,0x00,0x23,0x00,  /* 00001E90    "......#." */
+    0x00,0x00,0x01,0x00,0x5C,0x5F,0x53,0x42,  /* 00001E98    "....\_SB" */
+    0x2E,0x43,0x59,0x38,0x43,0x00,0x8C,0x20,  /* 00001EA0    ".CY8C.. " */
+    0x00,0x01,0x01,0x01,0x00,0x08,0x00,0x00,  /* 00001EA8    "........" */
+    0x00,0x00,0x00,0x00,0x17,0x00,0x00,0x19,  /* 00001EB0    "........" */
+    0x00,0x23,0x00,0x00,0x00,0x02,0x00,0x5C,  /* 00001EB8    ".#.....\" */
+    0x5F,0x53,0x42,0x2E,0x43,0x59,0x38,0x43,  /* 00001EC0    "_SB.CY8C" */
+    0x00,0x8C,0x20,0x00,0x01,0x01,0x01,0x00,  /* 00001EC8    ".. ....." */
+    0x08,0x00,0x00,0x00,0x00,0x00,0x00,0x17,  /* 00001ED0    "........" */
+    0x00,0x00,0x19,0x00,0x23,0x00,0x00,0x00,  /* 00001ED8    "....#..." */
+    0x03,0x00,0x5C,0x5F,0x53,0x42,0x2E,0x43,  /* 00001EE0    "..\_SB.C" */
+    0x59,0x38,0x43,0x00,0x8C,0x20,0x00,0x01,  /* 00001EE8    "Y8C.. .." */
+    0x01,0x01,0x00,0x08,0x00,0x00,0x00,0x00,  /* 00001EF0    "........" */
+    0x00,0x00,0x17,0x00,0x00,0x19,0x00,0x23,  /* 00001EF8    ".......#" */
+    0x00,0x00,0x00,0x04,0x00,0x5C,0x5F,0x53,  /* 00001F00    ".....\_S" */
+    0x42,0x2E,0x43,0x59,0x38,0x43,0x00,0x8C,  /* 00001F08    "B.CY8C.." */
+    0x20,0x00,0x01,0x01,0x01,0x00,0x08,0x00,  /* 00001F10    " ......." */
+    0x00,0x00,0x00,0x00,0x00,0x17,0x00,0x00,  /* 00001F18    "........" */
+    0x19,0x00,0x23,0x00,0x00,0x00,0x05,0x00,  /* 00001F20    "..#....." */
+    0x5C,0x5F,0x53,0x42,0x2E,0x43,0x59,0x38,  /* 00001F28    "\_SB.CY8" */
+    0x43,0x00,0x8C,0x20,0x00,0x01,0x01,0x01,  /* 00001F30    "C.. ...." */
+    0x00,0x08,0x00,0x00,0x00,0x00,0x00,0x00,  /* 00001F38    "........" */
+    0x17,0x00,0x00,0x19,0x00,0x23,0x00,0x00,  /* 00001F40    ".....#.." */
+    0x00,0x06,0x00,0x5C,0x5F,0x53,0x42,0x2E,  /* 00001F48    "...\_SB." */
+    0x43,0x59,0x38,0x43,0x00,0x8C,0x20,0x00,  /* 00001F50    "CY8C.. ." */
+    0x01,0x01,0x01,0x00,0x08,0x00,0x00,0x00,  /* 00001F58    "........" */
+    0x00,0x00,0x00,0x17,0x00,0x00,0x19,0x00,  /* 00001F60    "........" */
+    0x23,0x00,0x00,0x00,0x07,0x00,0x5C,0x5F,  /* 00001F68    "#.....\_" */
+    0x53,0x42,0x2E,0x43,0x59,0x38,0x43,0x00,  /* 00001F70    "SB.CY8C." */
+    0x8C,0x20,0x00,0x01,0x01,0x01,0x00,0x08,  /* 00001F78    ". ......" */
+    0x00,0x00,0x00,0x00,0x00,0x00,0x17,0x00,  /* 00001F80    "........" */
+    0x00,0x19,0x00,0x23,0x00,0x00,0x00,0x08,  /* 00001F88    "...#...." */
+    0x00,0x5C,0x5F,0x53,0x42,0x2E,0x43,0x59,  /* 00001F90    ".\_SB.CY" */
+    0x38,0x43,0x00,0x8C,0x20,0x00,0x01,0x01,  /* 00001F98    "8C.. ..." */
+    0x01,0x00,0x08,0x00,0x00,0x00,0x00,0x00,  /* 00001FA0    "........" */
+    0x00,0x17,0x00,0x00,0x19,0x00,0x23,0x00,  /* 00001FA8    "......#." */
+    0x00,0x00,0x09,0x00,0x5C,0x5F,0x53,0x42,  /* 00001FB0    "....\_SB" */
+    0x2E,0x43,0x59,0x38,0x43,0x00,0x8C,0x20,  /* 00001FB8    ".CY8C.. " */
+    0x00,0x01,0x01,0x01,0x00,0x08,0x00,0x00,  /* 00001FC0    "........" */
+    0x00,0x00,0x00,0x00,0x17,0x00,0x00,0x19,  /* 00001FC8    "........" */
+    0x00,0x23,0x00,0x00,0x00,0x0A,0x00,0x5C,  /* 00001FD0    ".#.....\" */
+    0x5F,0x53,0x42,0x2E,0x43,0x59,0x38,0x43,  /* 00001FD8    "_SB.CY8C" */
+    0x00,0x8C,0x20,0x00,0x01,0x01,0x01,0x00,  /* 00001FE0    ".. ....." */
+    0x08,0x00,0x00,0x00,0x00,0x00,0x00,0x17,  /* 00001FE8    "........" */
+    0x00,0x00,0x19,0x00,0x23,0x00,0x00,0x00,  /* 00001FF0    "....#..." */
+    0x0B,0x00,0x5C,0x5F,0x53,0x42,0x2E,0x43,  /* 00001FF8    "..\_SB.C" */
+    0x59,0x38,0x43,0x00,0x8C,0x20,0x00,0x01,  /* 00002000    "Y8C.. .." */
+    0x01,0x01,0x00,0x08,0x00,0x00,0x00,0x00,  /* 00002008    "........" */
+    0x00,0x00,0x17,0x00,0x00,0x19,0x00,0x23,  /* 00002010    ".......#" */
+    0x00,0x00,0x00,0x0C,0x00,0x5C,0x5F,0x53,  /* 00002018    ".....\_S" */
+    0x42,0x2E,0x43,0x59,0x38,0x43,0x00,0x8C,  /* 00002020    "B.CY8C.." */
+    0x20,0x00,0x01,0x01,0x01,0x00,0x08,0x00,  /* 00002028    " ......." */
+    0x00,0x00,0x00,0x00,0x00,0x17,0x00,0x00,  /* 00002030    "........" */
+    0x19,0x00,0x23,0x00,0x00,0x00,0x0D,0x00,  /* 00002038    "..#....." */
+    0x5C,0x5F,0x53,0x42,0x2E,0x43,0x59,0x38,  /* 00002040    "\_SB.CY8" */
+    0x43,0x00,0x8C,0x20,0x00,0x01,0x01,0x01,  /* 00002048    "C.. ...." */
+    0x00,0x08,0x00,0x00,0x00,0x00,0x00,0x00,  /* 00002050    "........" */
+    0x17,0x00,0x00,0x19,0x00,0x23,0x00,0x00,  /* 00002058    ".....#.." */
+    0x00,0x0E,0x00,0x5C,0x5F,0x53,0x42,0x2E,  /* 00002060    "...\_SB." */
+    0x43,0x59,0x38,0x43,0x00,0x8C,0x20,0x00,  /* 00002068    "CY8C.. ." */
+    0x01,0x01,0x01,0x00,0x08,0x00,0x00,0x00,  /* 00002070    "........" */
+    0x00,0x00,0x00,0x17,0x00,0x00,0x19,0x00,  /* 00002078    "........" */
+    0x23,0x00,0x00,0x00,0x0F,0x00,0x5C,0x5F,  /* 00002080    "#.....\_" */
+    0x53,0x42,0x2E,0x43,0x59,0x38,0x43,0x00,  /* 00002088    "SB.CY8C." */
+    0x8C,0x20,0x00,0x01,0x01,0x01,0x00,0x08,  /* 00002090    ". ......" */
+    0x00,0x00,0x00,0x00,0x00,0x00,0x17,0x00,  /* 00002098    "........" */
+    0x00,0x19,0x00,0x23,0x00,0x00,0x00,0x10,  /* 000020A0    "...#...." */
+    0x00,0x5C,0x5F,0x53,0x42,0x2E,0x43,0x59,  /* 000020A8    ".\_SB.CY" */
+    0x38,0x43,0x00,0x8C,0x20,0x00,0x01,0x01,  /* 000020B0    "8C.. ..." */
+    0x01,0x00,0x08,0x00,0x00,0x00,0x00,0x00,  /* 000020B8    "........" */
+    0x00,0x17,0x00,0x00,0x19,0x00,0x23,0x00,  /* 000020C0    "......#." */
+    0x00,0x00,0x11,0x00,0x5C,0x5F,0x53,0x42,  /* 000020C8    "....\_SB" */
+    0x2E,0x43,0x59,0x38,0x43,0x00,0x8C,0x20,  /* 000020D0    ".CY8C.. " */
+    0x00,0x01,0x01,0x01,0x00,0x08,0x00,0x00,  /* 000020D8    "........" */
+    0x00,0x00,0x00,0x00,0x17,0x00,0x00,0x19,  /* 000020E0    "........" */
+    0x00,0x23,0x00,0x00,0x00,0x12,0x00,0x5C,  /* 000020E8    ".#.....\" */
+    0x5F,0x53,0x42,0x2E,0x43,0x59,0x38,0x43,  /* 000020F0    "_SB.CY8C" */
+    0x00,0x8C,0x20,0x00,0x01,0x01,0x01,0x00,  /* 000020F8    ".. ....." */
+    0x08,0x00,0x00,0x00,0x00,0x00,0x00,0x17,  /* 00002100    "........" */
+    0x00,0x00,0x19,0x00,0x23,0x00,0x00,0x00,  /* 00002108    "....#..." */
+    0x13,0x00,0x5C,0x5F,0x53,0x42,0x2E,0x43,  /* 00002110    "..\_SB.C" */
+    0x59,0x38,0x43,0x00,0x8C,0x20,0x00,0x01,  /* 00002118    "Y8C.. .." */
+    0x01,0x01,0x00,0x08,0x00,0x00,0x00,0x00,  /* 00002120    "........" */
+    0x00,0x00,0x17,0x00,0x00,0x19,0x00,0x23,  /* 00002128    ".......#" */
+    0x00,0x00,0x00,0x14,0x00,0x5C,0x5F,0x53,  /* 00002130    ".....\_S" */
+    0x42,0x2E,0x43,0x59,0x38,0x43,0x00,0x8C,  /* 00002138    "B.CY8C.." */
+    0x20,0x00,0x01,0x01,0x01,0x00,0x08,0x00,  /* 00002140    " ......." */
+    0x00,0x00,0x00,0x00,0x00,0x17,0x00,0x00,  /* 00002148    "........" */
+    0x19,0x00,0x23,0x00,0x00,0x00,0x15,0x00,  /* 00002150    "..#....." */
+    0x5C,0x5F,0x53,0x42,0x2E,0x43,0x59,0x38,  /* 00002158    "\_SB.CY8" */
+    0x43,0x00,0x8C,0x20,0x00,0x01,0x01,0x01,  /* 00002160    "C.. ...." */
+    0x00,0x08,0x00,0x00,0x00,0x00,0x00,0x00,  /* 00002168    "........" */
+    0x17,0x00,0x00,0x19,0x00,0x23,0x00,0x00,  /* 00002170    ".....#.." */
+    0x00,0x16,0x00,0x5C,0x5F,0x53,0x42,0x2E,  /* 00002178    "...\_SB." */
+    0x43,0x59,0x38,0x43,0x00,0x8C,0x20,0x00,  /* 00002180    "CY8C.. ." */
+    0x01,0x01,0x01,0x00,0x08,0x00,0x00,0x00,  /* 00002188    "........" */
+    0x00,0x00,0x00,0x17,0x00,0x00,0x19,0x00,  /* 00002190    "........" */
+    0x23,0x00,0x00,0x00,0x17,0x00,0x5C,0x5F,  /* 00002198    "#.....\_" */
+    0x53,0x42,0x2E,0x43,0x59,0x38,0x43,0x00,  /* 000021A0    "SB.CY8C." */
+    0x8C,0x20,0x00,0x01,0x01,0x01,0x00,0x08,  /* 000021A8    ". ......" */
+    0x00,0x00,0x00,0x00,0x00,0x00,0x17,0x00,  /* 000021B0    "........" */
+    0x00,0x19,0x00,0x23,0x00,0x00,0x00,0x18,  /* 000021B8    "...#...." */
+    0x00,0x5C,0x5F,0x53,0x42,0x2E,0x43,0x59,  /* 000021C0    ".\_SB.CY" */
+    0x38,0x43,0x00,0x8C,0x20,0x00,0x01,0x01,  /* 000021C8    "8C.. ..." */
+    0x01,0x00,0x08,0x00,0x00,0x00,0x00,0x00,  /* 000021D0    "........" */
+    0x00,0x17,0x00,0x00,0x19,0x00,0x23,0x00,  /* 000021D8    "......#." */
+    0x00,0x00,0x19,0x00,0x5C,0x5F,0x53,0x42,  /* 000021E0    "....\_SB" */
+    0x2E,0x43,0x59,0x38,0x43,0x00,0x8C,0x20,  /* 000021E8    ".CY8C.. " */
+    0x00,0x01,0x01,0x01,0x00,0x08,0x00,0x00,  /* 000021F0    "........" */
+    0x00,0x00,0x00,0x00,0x17,0x00,0x00,0x19,  /* 000021F8    "........" */
+    0x00,0x23,0x00,0x00,0x00,0x1A,0x00,0x5C,  /* 00002200    ".#.....\" */
+    0x5F,0x53,0x42,0x2E,0x43,0x59,0x38,0x43,  /* 00002208    "_SB.CY8C" */
+    0x00,0x8C,0x20,0x00,0x01,0x01,0x01,0x00,  /* 00002210    ".. ....." */
+    0x08,0x00,0x00,0x00,0x00,0x00,0x00,0x17,  /* 00002218    "........" */
+    0x00,0x00,0x19,0x00,0x23,0x00,0x00,0x00,  /* 00002220    "....#..." */
+    0x1B,0x00,0x5C,0x5F,0x53,0x42,0x2E,0x43,  /* 00002228    "..\_SB.C" */
+    0x59,0x38,0x43,0x00,0x8C,0x20,0x00,0x01,  /* 00002230    "Y8C.. .." */
+    0x01,0x01,0x00,0x08,0x00,0x00,0x00,0x00,  /* 00002238    "........" */
+    0x00,0x00,0x17,0x00,0x00,0x19,0x00,0x23,  /* 00002240    ".......#" */
+    0x00,0x00,0x00,0x1C,0x00,0x5C,0x5F,0x53,  /* 00002248    ".....\_S" */
+    0x42,0x2E,0x43,0x59,0x38,0x43,0x00,0x8C,  /* 00002250    "B.CY8C.." */
+    0x20,0x00,0x01,0x01,0x01,0x00,0x08,0x00,  /* 00002258    " ......." */
+    0x00,0x00,0x00,0x00,0x00,0x17,0x00,0x00,  /* 00002260    "........" */
+    0x19,0x00,0x23,0x00,0x00,0x00,0x1D,0x00,  /* 00002268    "..#....." */
+    0x5C,0x5F,0x53,0x42,0x2E,0x43,0x59,0x38,  /* 00002270    "\_SB.CY8" */
+    0x43,0x00,0x8C,0x20,0x00,0x01,0x01,0x01,  /* 00002278    "C.. ...." */
+    0x00,0x08,0x00,0x00,0x00,0x00,0x00,0x00,  /* 00002280    "........" */
+    0x17,0x00,0x00,0x19,0x00,0x23,0x00,0x00,  /* 00002288    ".....#.." */
+    0x00,0x1E,0x00,0x5C,0x5F,0x53,0x42,0x2E,  /* 00002290    "...\_SB." */
+    0x43,0x59,0x38,0x43,0x00,0x8C,0x20,0x00,  /* 00002298    "CY8C.. ." */
+    0x01,0x01,0x01,0x00,0x08,0x00,0x00,0x00,  /* 000022A0    "........" */
+    0x00,0x00,0x00,0x17,0x00,0x00,0x19,0x00,  /* 000022A8    "........" */
+    0x23,0x00,0x00,0x00,0x1F,0x00,0x5C,0x5F,  /* 000022B0    "#.....\_" */
+    0x53,0x42,0x2E,0x43,0x59,0x38,0x43,0x00,  /* 000022B8    "SB.CY8C." */
+    0x8C,0x20,0x00,0x01,0x01,0x01,0x00,0x08,  /* 000022C0    ". ......" */
+    0x00,0x00,0x00,0x00,0x00,0x00,0x17,0x00,  /* 000022C8    "........" */
+    0x00,0x19,0x00,0x23,0x00,0x00,0x00,0x20,  /* 000022D0    "...#... " */
+    0x00,0x5C,0x5F,0x53,0x42,0x2E,0x43,0x59,  /* 000022D8    ".\_SB.CY" */
+    0x38,0x43,0x00,0x8C,0x20,0x00,0x01,0x01,  /* 000022E0    "8C.. ..." */
+    0x01,0x00,0x08,0x00,0x00,0x00,0x00,0x00,  /* 000022E8    "........" */
+    0x00,0x17,0x00,0x00,0x19,0x00,0x23,0x00,  /* 000022F0    "......#." */
+    0x00,0x00,0x21,0x00,0x5C,0x5F,0x53,0x42,  /* 000022F8    "..!.\_SB" */
+    0x2E,0x43,0x59,0x38,0x43,0x00,0x8C,0x20,  /* 00002300    ".CY8C.. " */
+    0x00,0x01,0x01,0x01,0x00,0x08,0x00,0x00,  /* 00002308    "........" */
+    0x00,0x00,0x00,0x00,0x17,0x00,0x00,0x19,  /* 00002310    "........" */
+    0x00,0x23,0x00,0x00,0x00,0x22,0x00,0x5C,  /* 00002318    ".#...".\" */
+    0x5F,0x53,0x42,0x2E,0x43,0x59,0x38,0x43,  /* 00002320    "_SB.CY8C" */
+    0x00,0x8C,0x20,0x00,0x01,0x01,0x01,0x00,  /* 00002328    ".. ....." */
+    0x08,0x00,0x00,0x00,0x00,0x00,0x00,0x17,  /* 00002330    "........" */
+    0x00,0x00,0x19,0x00,0x23,0x00,0x00,0x00,  /* 00002338    "....#..." */
+    0x23,0x00,0x5C,0x5F,0x53,0x42,0x2E,0x43,  /* 00002340    "#.\_SB.C" */
+    0x59,0x38,0x43,0x00,0x8C,0x20,0x00,0x01,  /* 00002348    "Y8C.. .." */
+    0x01,0x01,0x00,0x08,0x00,0x00,0x00,0x00,  /* 00002350    "........" */
+    0x00,0x00,0x17,0x00,0x00,0x19,0x00,0x23,  /* 00002358    ".......#" */
+    0x00,0x00,0x00,0x24,0x00,0x5C,0x5F,0x53,  /* 00002360    "...$.\_S" */
+    0x42,0x2E,0x43,0x59,0x38,0x43,0x00,0x8C,  /* 00002368    "B.CY8C.." */
+    0x20,0x00,0x01,0x01,0x01,0x00,0x08,0x00,  /* 00002370    " ......." */
+    0x00,0x00,0x00,0x00,0x00,0x17,0x00,0x00,  /* 00002378    "........" */
+    0x19,0x00,0x23,0x00,0x00,0x00,0x25,0x00,  /* 00002380    "..#...%." */
+    0x5C,0x5F,0x53,0x42,0x2E,0x43,0x59,0x38,  /* 00002388    "\_SB.CY8" */
+    0x43,0x00,0x8C,0x20,0x00,0x01,0x01,0x01,  /* 00002390    "C.. ...." */
+    0x00,0x08,0x00,0x00,0x00,0x00,0x00,0x00,  /* 00002398    "........" */
+    0x17,0x00,0x00,0x19,0x00,0x23,0x00,0x00,  /* 000023A0    ".....#.." */
+    0x00,0x26,0x00,0x5C,0x5F,0x53,0x42,0x2E,  /* 000023A8    ".&.\_SB." */
+    0x43,0x59,0x38,0x43,0x00,0x8C,0x20,0x00,  /* 000023B0    "CY8C.. ." */
+    0x01,0x01,0x01,0x00,0x08,0x00,0x00,0x00,  /* 000023B8    "........" */
+    0x00,0x00,0x00,0x17,0x00,0x00,0x19,0x00,  /* 000023C0    "........" */
+    0x23,0x00,0x00,0x00,0x27,0x00,0x5C,0x5F,  /* 000023C8    "#...'.\_" */
+    0x53,0x42,0x2E,0x43,0x59,0x38,0x43,0x00,  /* 000023D0    "SB.CY8C." */
+    0x79,0x00,0xA4,0x52,0x42,0x55,0x46,0x14,  /* 000023D8    "y..RBUF." */
+    0x15,0x5F,0x53,0x54,0x41,0x00,0xA0,0x0B,  /* 000023E0    "._STA..." */
+    0x92,0x93,0x50,0x54,0x59,0x50,0x0A,0x06,  /* 000023E8    "..PTYP.." */
+    0xA4,0x00,0xA4,0x0A,0x0F                  /* 000023F0    "....."    */
+};
-- 
2.5.0

